# acc

Version: 1.3.3

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Jaejoon Song <jjsong2@mdanderson.org>’
    
    Insufficient package version (submitted: 1.3.3, existing: 1.3.3)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# ACNE

Version: 0.8.1

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Henrik Bengtsson <henrikb@braju.com>’
    
    Insufficient package version (submitted: 0.8.1, existing: 0.8.1)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# acnr

Version: 1.0.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Morgane Pierre-Jean <morgane.pierrejean@genopole.cnrs.fr>’
    
    Insufficient package version (submitted: 1.0.0, existing: 1.0.0)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# affxparser

Version: 1.48.0

## In both

*   checking compiled code ... WARNING
    ```
    ...
          ‘fusion_sdk/file/BPMAPFileWriter.o’
      Found ‘exit’, possibly from ‘exit’ (C)
        Objects: ‘fusion_sdk/util/Err.o’, ‘R_affx_clf_pgf_parser.o’
      Found ‘printf’, possibly from ‘printf’ (C)
        Object: ‘fusion_sdk/file/TsvFile/TsvFile.o’
      Found ‘putchar’, possibly from ‘putchar’ (C)
        Object: ‘fusion_sdk/file/TsvFile/TsvFile.o’
      Found ‘puts’, possibly from ‘printf’ (C), ‘puts’ (C)
        Object: ‘fusion_sdk/file/TsvFile/TsvFile.o’
      Found ‘rand’, possibly from ‘rand’ (C)
        Objects: ‘fusion_sdk/calvin_files/utils/src/AffymetrixGuid.o’,
          ‘fusion_sdk/util/Verbose.o’
      Found ‘srand’, possibly from ‘srand’ (C)
        Object: ‘fusion_sdk/calvin_files/utils/src/AffymetrixGuid.o’
      Found ‘stdout’, possibly from ‘stdout’ (C)
        Object: ‘fusion_sdk/util/Util.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor the system RNG.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Kasper Daniel Hansen <kasperdanielhansen@gmail.com>’
    
    Package duplicated from https://bioconductor.org/packages/3.5/bioc
    
    Found the following (possibly) invalid URLs:
      URL: http://www.affymetrix.com/support/datasets.affx (moved to https://www.thermofisher.com/us/en/home/life-science/microarray-analysis/microarray-data-analysis/microarray-analysis-sample-data.html)
        From: man/affxparser-package.Rd
        Status: 503
        Message: Service Unavailable
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking for portable file names ... NOTE
    ```
    Found the following non-portable file paths:
      affxparser/src/fusion_sdk/calvin_files/writers/src/CalvinCHPQuantificationDetectionFileBufferWriter.cpp
      affxparser/src/fusion_sdk/calvin_files/writers/src/CalvinCHPQuantificationDetectionFileBufferWriter.h
    
    Tarballs are only required to store paths of up to 100 bytes and cannot
    store those of more than 256 bytes, with restrictions including to 100
    bytes for the final component.
    See section ‘Package structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 14.5Mb
      sub-directories of 1Mb or more:
        libs  13.9Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘affy’
    ```

# alphahull

Version: 2.1

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Beatriz Pateiro-Lopez <beatriz.pateiro@usc.es>’
    
    Insufficient package version (submitted: 2.1, existing: 2.1)
    
    CRAN repository db overrides:
      License_restricts_use: yes
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# arc

Version: 1.1.2

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Tomas Kliegr <kliegr@gmail.com>’
    
    Insufficient package version (submitted: 1.1.2, existing: 1.1.2)
    
    This build time stamp is over a month old.
    ```

# aroma.affymetrix

Version: 3.1.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Henrik Bengtsson <henrikb@braju.com>’
    
    Insufficient package version (submitted: 3.1.0, existing: 3.1.0)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# aroma.apd

Version: 0.6.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Henrik Bengtsson <henrikb@braju.com>’
    
    Insufficient package version (submitted: 0.6.0, existing: 0.6.0)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# aroma.cn

Version: 1.6.1

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Henrik Bengtsson <henrikb@braju.com>’
    
    Insufficient package version (submitted: 1.6.1, existing: 1.6.1)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# aroma.core

Version: 3.1.1

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Henrik Bengtsson <henrikb@braju.com>’
    
    Insufficient package version (submitted: 3.1.1, existing: 3.1.1)
    
    Suggests or Enhances not in mainstream repositories:
      sfit, expectile, HaarSeg, mpcbs
    Availability using Additional_repositories specification:
      sfit        yes   https://henrikbengtsson.github.io/drat
      expectile   yes   http://r-forge.r-project.org          
      HaarSeg     yes   http://r-forge.r-project.org          
      mpcbs       yes   http://r-forge.r-project.org          
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘sfit’ ‘expectile’ ‘HaarSeg’ ‘mpcbs’
    ```

# aroma.light

Version: 3.6.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Henrik Bengtsson <henrikb@braju.com>’
    
    Package duplicated from https://bioconductor.org/packages/3.5/bioc
    
    CRAN repository db overrides:
      X-CRAN-Comment: This package is now available from Bioconductor only,
        see
        <http://www.bioconductor.org/packages/release/bioc/html/aroma.light.html>.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      inst/rsp/.rspPlugins
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    
    CRAN-pack does not know about
      inst/rsp/.rspPlugins
    ```

*   checking top-level files ... NOTE
    ```
    Non-standard file/directory found at top level:
      ‘revdep’
    ```

# awsjavasdk

Version: 0.2.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Russell Pierce <russell.pierce@zapier.com>’
    
    Insufficient package version (submitted: 0.2.0, existing: 0.2.0)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch http://img.shields.io/badge/license-GPL%20%28%3E=%202%29-brightgreen.svg?style=flat
    TlsExceptionHostPort (HandshakeFailed Error_EOF) "img.shields.io" 80
    ```

# babel

Version: 0.3-0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Adam B. Olshen <adam.olshen@ucsf.edu>’
    
    Insufficient package version (submitted: 0.3.0, existing: 0.3.0)
    
    This build time stamp is over a month old.
    ```

# bedr

Version: 1.0.4

## In both

*   checking examples ... ERROR
    ```
    ...
       -0, --zero-based           coordinates are zero-based
       -b, --begin INT            column number for region start [4]
       -c, --comment CHAR         skip comment lines starting with CHAR [null]
       -C, --csi                  generate CSI index for VCF (default is TBI)
       -e, --end INT              column number for region end (if no end, set INT to -b) [5]
       -f, --force                overwrite existing index without asking
       -m, --min-shift INT        set minimal interval size for CSI indices to 2^INT [14]
       -p, --preset STR           gff, bed, sam, vcf
       -s, --sequence INT         column number for sequence names (suppressed by -p) [1]
       -S, --skip-lines INT       skip first INT lines [0]
    
    Querying and other options:
       -h, --print-header         print also the header lines
       -H, --only-header          print only the header lines
       -l, --list-chroms          list chromosome names
       -r, --reheader FILE        replace the header with the content of FILE
       -R, --regions FILE         restrict to regions listed in the file
       -T, --targets FILE         similar to -R but streams rather than index-jumps
    
       ERROR: Looks like tabix had a problemError in tabix(query.regions, cosmic.vcf.example, check.chr = FALSE) : 
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/test.all.R’ failed.
    Last 13 lines of output:
      
      
      2. Error: check tabix (@test.tabix.R#38) ---------------------------------------
      
      1: expect_equal(nrow(tabix(a.nochr, vcf, verbose = T, check.chr = FALSE)), NULL) at /home/hb/repositories/R.utils/revdep/checks/bedr/new/bedr.Rcheck/bedr/tests/test.tabix.R:38
      2: compare(object, expected, ...)
      3: nrow(tabix(a.nochr, vcf, verbose = T, check.chr = FALSE))
      4: tabix(a.nochr, vcf, verbose = T, check.chr = FALSE)
      5: stop()
      
      DONE ===========================================================================
      Error: Test failures
      In addition: Warning message:
      Placing tests in `inst/tests/` is deprecated. Please use `tests/testthat/` instead 
      Execution halted
    ```

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Paul C. Boutros <Paul.Boutros@oicr.on.ca>’
    
    Insufficient package version (submitted: 1.0.4, existing: 1.0.4)
    
    Days since last update: 5
    ```

# bigstep

Version: 0.7.4

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Piotr Szulc <piotr.michal.szulc@gmail.com>’
    
    Insufficient package version (submitted: 0.7.4, existing: 0.7.4)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# BioInstaller

Version: 0.2.1

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Jianfeng Li <lee_jianfeng@sjtu.edu.cn>’
    
    Insufficient package version (submitted: 0.2.1, existing: 0.2.1)
    ```

# bsseq

Version: 1.12.2

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Kasper Daniel Hansen <kasperdanielhansen@gmail.com>’
    
    Package duplicated from https://bioconductor.org/packages/3.5/bioc
    
    Found the following (possibly) invalid URLs:
      URL: http://www.bioinformatics.bbsrc.ac.uk/projects/bismark/RELEASE_NOTES.txt
        From: man/read.bismark.Rd
        Status: 403
        Message: Forbidden
    
    The Title field should be in title case, current version then in title case:
    ‘Analyze, manage and store bisulfite sequencing data’
    ‘Analyze, Manage and Store Bisulfite Sequencing Data’
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘bsseqData’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘HDF5Array’
      All declared Imports should be used.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘bsseqData’
    ```

# calmate

Version: 0.12.1

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Henrik Bengtsson <henrikb@braju.com>’
    
    Insufficient package version (submitted: 0.12.1, existing: 0.12.1)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# Causata

Version: 4.2-0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Justin Hemann <justinh@causata.com>’
    
    Insufficient package version (submitted: 4.2.0, existing: 4.2.0)
    
    The Title field should be in title case, current version then in title case:
    ‘Analysis utilities for binary classification and Causata users.’
    ‘Analysis Utilities for Binary Classification and Causata Users.’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

*   checking top-level files ... NOTE
    ```
    Non-standard file/directory found at top level:
      ‘integration_tests’
    ```

# cellbaseR

Version: 1.0.0

## Newly broken

*   R CMD check timed out
    

## Newly fixed

*   checking examples ... ERROR
    ```
    ...
    Use 'rbind_pages' instead.
    See help("Deprecated")
    Warning: 'rbind.pages' is deprecated.
    Use 'rbind_pages' instead.
    See help("Deprecated")
    Warning: 'rbind.pages' is deprecated.
    Use 'rbind_pages' instead.
    See help("Deprecated")
    Warning: 'rbind.pages' is deprecated.
    Use 'rbind_pages' instead.
    See help("Deprecated")
    Warning: 'rbind.pages' is deprecated.
    Use 'rbind_pages' instead.
    See help("Deprecated")
    Warning: 'rbind.pages' is deprecated.
    Use 'rbind_pages' instead.
    See help("Deprecated")
    Error: lexical error: invalid char in json text.
                                           <html><body><h1>504 Gateway Tim
                         (right here) ------^
    Execution halted
    ```

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Mohammed OE Abdallah <melsiddieg@gmail.com>’
    
    Package duplicated from https://bioconductor.org/packages/3.5/bioc
    
    Unknown, possibly mis-spelled, fields in DESCRIPTION:
      ‘Lazy’
    
    The Title field should be in title case, current version then in title case:
    ‘Querying annotation data from the high performance Cellbase web services’
    ‘Querying Annotation Data from the High Performance Cellbase Web Services’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# ChemoSpec

Version: 4.4.85

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Bryan A. Hanson <hanson@depauw.edu>’
    
    Insufficient package version (submitted: 4.4.85, existing: 4.4.85)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘chemometrics’, ‘mvoutlier’
    ```

# ChocoLattes

Version: 0.1.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Felipe Campelo <fcampelo@ufmg.br>’
    
    Insufficient package version (submitted: 0.1.0, existing: 0.1.0)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘knitr’ ‘tools’
      All declared Imports should be used.
    ```

# civis

Version: 1.0.2

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Patrick Miller <pmiller@civisanalytics.com>’
    
    Insufficient package version (submitted: 1.0.2, existing: 1.0.2)
    ```

# CNEr

Version: 1.12.1

## In both

*   checking compiled code ... WARNING
    ```
    File ‘CNEr/libs/CNEr.so’:
      Found ‘abort’, possibly from ‘abort’ (C)
        Object: ‘ucsc/errabort.o’
      Found ‘exit’, possibly from ‘exit’ (C)
        Objects: ‘ucsc/errabort.o’, ‘ucsc/pipeline.o’
      Found ‘puts’, possibly from ‘printf’ (C), ‘puts’ (C)
        Object: ‘ucsc/pipeline.o’
      Found ‘rand’, possibly from ‘rand’ (C)
        Object: ‘ucsc/obscure.o’
      Found ‘stderr’, possibly from ‘stderr’ (C)
        Objects: ‘ucsc/axt.o’, ‘ucsc/errabort.o’, ‘ucsc/obscure.o’,
          ‘ucsc/verbose.o’, ‘ucsc/os.o’
      Found ‘stdout’, possibly from ‘stdout’ (C)
        Objects: ‘ucsc/common.o’, ‘ucsc/errabort.o’, ‘ucsc/verbose.o’,
          ‘ucsc/os.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor the system RNG.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Ge Tan <ge_tan@live.com>’
    
    Package duplicated from https://bioconductor.org/packages/3.5/bioc
    
    No package encoding and non-ASCII characters in the following R files:
      R/utils.R
        222: ### savefig: <e2><80><98>savefig<e2><80><99> saves figures to files with minimal surrounding white
        223: ### space, suitable for inclusion in books and reports. <e2><80><98>savefig<e2><80><99> is
        225: ### supported are eps, pdf and jpg. Default file format for <e2><80><98>savefig<e2><80><99>
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    
    Size of tarball: 9326126 bytes
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 18.3Mb
      sub-directories of 1Mb or more:
        extdata  15.9Mb
        libs      1.1Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘BiocGenerics:::replaceSlots’ ‘S4Vectors:::make_zero_col_DataFrame’
      See the note in ?`:::` about the use of this operator.
    ```

# CollapsABEL

Version: 0.10.11

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Kaiyin Zhong <kindlychung@gmail.com>’
    
    Insufficient package version (submitted: 0.10.11, existing: 0.10.11)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# countyfloods

Version: 0.0.2

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Rod Lammers <rodlammers@gmail.com>’
    
    Insufficient package version (submitted: 0.0.2, existing: 0.0.2)
    
    This build time stamp is over a month old.
    ```

# DeepBlueR

Version: 1.2.10

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Felipe Albrecht <felipe.albrecht@mpi-inf.mpg.de>, Markus List <markus.list@mpi-inf.mpg.de>’
    
    The maintainer field is invalid or specifies more than one person
    
    Package duplicated from https://bioconductor.org/packages/3.5/bioc
    
    The Title field is just the package name: provide a real title.
    
    This build time stamp is over a month old.
    ```

*   checking top-level files ... NOTE
    ```
    Non-standard file/directory found at top level:
      ‘logo.png’
    ```

# DEGraph

Version: 1.28.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Laurent Jacob <laurent.jacob@gmail.com>’
    
    Package duplicated from https://bioconductor.org/packages/3.5/bioc
    
    The Title field should be in title case, current version then in title case:
    ‘Two-sample tests on a graph’
    ‘Two-Sample Tests on a Graph’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    
    CRAN-pack does not know about
      .BBSoptions
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Packages listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘graph’ ‘KEGGgraph’ ‘lattice’ ‘RBGL’ ‘rrcov’ ‘Rgraphviz’ ‘NCIgraph’
    A package should be listed in only one of these fields.
    ```

*   checking top-level files ... NOTE
    ```
    Non-standard file/directory found at top level:
      ‘incl’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘R.utils’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    Unexported object imported by a ':::' call: ‘graph:::getIndices’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    twoSampleFromGraph: no visible global function definition for ‘rnorm’
    writeAdjacencyMatrix2KGML: no visible binding for global variable
      ‘Arguments’
    writeAdjacencyMatrix2KGML: no visible global function definition for
      ‘pushState’
    writeAdjacencyMatrix2KGML: no visible global function definition for
      ‘popState’
    writeAdjacencyMatrix2KGML : <anonymous>: no visible global function
      definition for ‘na.omit’
    Undefined global functions or variables:
      Arguments ProgressBar as enter exit heat.colors increase na.omit
      phyper pnorm popState pushState quantile reset rnorm str
      translateNCI2GeneID var
    Consider adding
      importFrom("grDevices", "heat.colors")
      importFrom("methods", "as")
      importFrom("stats", "na.omit", "phyper", "pnorm", "quantile", "rnorm",
                 "var")
      importFrom("utils", "str")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# demi

Version: 1.1.2

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Sten Ilmjarv <sten.ilmjarv@gmail.com>’
    
    Insufficient package version (submitted: 1.1.2, existing: 1.1.2)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    demi.wilcox.test.fast: no visible global function definition for
      ‘wilcox.test’
    demiequal: no visible global function definition for ‘wilcox.test’
    wprob: no visible global function definition for ‘combn’
    diffexp,DEMIDiff: no visible global function definition for ‘median’
    diffexp,DEMIDiff: no visible global function definition for ‘p.adjust’
    loadAnnotation,DEMIExperiment-environment: no visible global function
      definition for ‘data’
    loadBlat,DEMIExperiment-environment: no visible global function
      definition for ‘data’
    loadCytoband,DEMIExperiment-environment: no visible global function
      definition for ‘data’
    loadPathway,DEMIExperiment-environment: no visible global function
      definition for ‘data’
    Undefined global functions or variables:
      combn data dhyper median p.adjust t.test wilcox.test write.table
    Consider adding
      importFrom("stats", "dhyper", "median", "p.adjust", "t.test",
                 "wilcox.test")
      importFrom("utils", "combn", "data", "write.table")
    to your NAMESPACE file.
    ```

# downsize

Version: 0.2.2

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘William Michael Landau <will.landau@gmail.com>’
    
    Insufficient package version (submitted: 0.2.2, existing: 0.2.2)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# dplR

Version: 1.6.6

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    ...
    Insufficient package version (submitted: 1.6.6, existing: 1.6.6)
    
    Found the following (possibly) invalid URLs:
      URL: http://dendrolab.org/
        From: man/bai.in.Rd
              man/bai.out.Rd
              man/cms.Rd
              man/ffcsaps.Rd
              man/gp.d2pith.Rd
              man/gp.dbh.Rd
              man/gp.po.Rd
              man/gp.rwl.Rd
              man/rcs.Rd
        Status: Error
        Message: libcurl error code 60:
        	SSL certificate problem: unable to get local issuer certificate
        	(Status without verification: OK)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# drake

Version: 4.3.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘William Michael Landau <will.landau@lilly.com>’
    
    Insufficient package version (submitted: 4.3.0, existing: 4.3.0)
    
    Days since last update: 0
    ```

# DupChecker

Version: 1.14.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘"Quanhu SHENG" <shengqh@gmail.com>’
    
    Package duplicated from https://bioconductor.org/packages/3.5/bioc
    
    The Title field should be in title case, current version then in title case:
    ‘a package for checking high-throughput genomic data redundancy in meta-analysis’
    ‘a Package for Checking High-Throughput Genomic Data Redundancy in Meta-Analysis’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking R code for possible problems ... NOTE
    ```
    arrayExpressDownload: no visible global function definition for
      ‘download.file’
    geoDownload: no visible global function definition for ‘download.file’
    geoDownload: no visible global function definition for ‘untar’
    getFtpFilenames: no visible global function definition for ‘read.table’
    Undefined global functions or variables:
      download.file read.table untar
    Consider adding
      importFrom("utils", "download.file", "read.table", "untar")
    to your NAMESPACE file.
    ```

# edgar

Version: 1.0.9

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Gunratan Lonare <lonare.gunratan@gmail.com>’
    
    Insufficient package version (submitted: 1.0.9, existing: 1.0.9)
    
    Found the following (possibly) invalid URLs:
      URL: https://www.sec.gov/Archives/edgar/full-index/
        From: man/getMasterIndex.Rd
        Status: 404
        Message: Not Found
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# eegc

Version: 1.2.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 28-29 (eegc.Rnw) 
    Error: processing vignette 'eegc.Rnw' failed with diagnostics:
    there is no package called 'BiocStyle'
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Xiaoyuan Zhou <zhouxiaoyuan@picb.ac.cn>’
    
    Package duplicated from https://bioconductor.org/packages/3.5/bioc
    
    The Title field should be in title case, current version then in title case:
    ‘Engineering Evaluation by Gene Categorization (eegc)’
    ‘Engineering Evaluation by Gene Categorization (Eegc)’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    
    Size of tarball: 9344858 bytes
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 11.1Mb
      sub-directories of 1Mb or more:
        data  10.5Mb
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' import not declared from: ‘BiocStyle’
    ```

# EEM

Version: 1.1.1

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Vipavee Trivittayasil <vipavee.tri@gmail.com>’
    
    Insufficient package version (submitted: 1.1.1, existing: 1.1.1)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# eiCompare

Version: 2.1

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Loren Collingwood <loren.collingwood@ucr.edu>’
    
    Insufficient package version (submitted: 2.1, existing: 2.1)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# ELMER

Version: 1.6.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Simon Coetzee <Simon.Coetzee@cshs.org>’
    
    Package duplicated from https://bioconductor.org/packages/3.5/bioc
    
    Uses the superseded package: ‘snow’
    
    Found the following (possibly) invalid URLs:
      URL: https://tcga-data.nci.nih.gov/tcga/tcgaHome2.jsp
        From: man/get450K.Rd
              man/getClinic.Rd
              man/getRNAseq.Rd
              man/getTCGA.Rd
        Status: Error
        Message: libcurl error code 47:
        	Maximum (20) redirects followed
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    This build time stamp is over a month old.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘ELMER.data’ ‘Homo.sapiens’
      ‘IlluminaHumanMethylation450kanno.ilmn12.hg19’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

# eudysbiome

Version: 1.6.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Xiaoyuan Zhou <zhouxiaoyuan@picb.ac.cn>’
    
    Package duplicated from https://bioconductor.org/packages/3.5/bioc
    
    The Title field should be in title case, current version then in title case:
    ‘Cartesian plot and contingency test on 16S Microbial data’
    ‘Cartesian Plot and Contingency Test on 16s Microbial Data’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# evclust

Version: 1.0.3

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Thierry Denoeux <tdenoeux@utc.fr>’
    
    Insufficient package version (submitted: 1.0.3, existing: 1.0.3)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# ezknitr

Version: 0.6

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Dean Attali <daattali@gmail.com>’
    
    Insufficient package version (submitted: 0.6, existing: 0.6)
    
    This build time stamp is over a month old.
    ```

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch http://ropensci.org/public_images/github_footer.png
    TlsExceptionHostPort (HandshakeFailed Error_EOF) "ropensci.org" 80
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    pandoc: Could not fetch http://ropensci.org/public_images/github_footer.png
    TlsExceptionHostPort (HandshakeFailed Error_EOF) "ropensci.org" 80
    Error: processing vignette 'ezknitr.Rmd' failed with diagnostics:
    pandoc document conversion failed with error 67
    Execution halted
    ```

# FGNet

Version: 3.10.0

## In both

*   checking examples ... ERROR
    ```
    ...
    APC2    1   1   1   1   0   0   0   0   0   0   0   0   0   0    0    0   0   0
    APC4    1   1   1   1   0   0   0   0   0   0   0   0   0   0    0    0   0   0
    APC5    1   1   1   0   1   0   0   0   0   0   0   0   0   0    0    0   0   0
          6.3 6.4 6.5 6.6 6.7
    ADA2    0   0   0   0   0
    APC1    0   0   0   0   0
    APC11   0   0   0   0   0
    APC2    0   0   0   1   0
    APC4    0   0   0   0   0
    APC5    0   0   0   0   0
    > 
    > functionalNetwork(incidMat)
    >     
    > # Term-based network
    > incidMatTerms <- fea2incidMat(results, key="Terms")
    > functionalNetwork(incidMatTerms, plotOutput="dynamic")
    Warning: no DISPLAY variable so Tk is not available
    Error in structure(.External(.C_dotTclObjv, objv), class = "tclObj") : 
      [tcl] invalid command name "font".
    Calls: functionalNetwork ... .tkplot.convert.font -> <Anonymous> -> tcl -> .Tcl.objv -> structure
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Sara Aibar <saibar@usal.es>’
    
    Package duplicated from https://bioconductor.org/packages/3.5/bioc
    
    The Title field should be in title case, current version then in title case:
    ‘Functional Gene Networks derived from biological enrichment analyses’
    ‘Functional Gene Networks Derived from Biological Enrichment Analyses’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    
    Size of tarball: 6584956 bytes
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.5Mb
      sub-directories of 1Mb or more:
        doc   7.0Mb
    ```

# ForecastFramework

Version: 0.9.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Joshua Kaminsky <jkaminsky@jhu.edu>’
    
    Insufficient package version (submitted: 0.9.0, existing: 0.9.0)
    
    This build time stamp is over a month old.
    ```

# freesurfer

Version: 1.6.1

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘John Muschelli <muschellij2@gmail.com>’
    
    Insufficient package version (submitted: 1.6.1, existing: 1.6.1)
    ```

# fslr

Version: 2.12

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘John Muschelli <muschellij2@gmail.com>’
    
    Insufficient package version (submitted: 2.12, existing: 2.12)
    
    This build time stamp is over a month old.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘R.utils’ ‘grDevices’ ‘graphics’ ‘matrixStats’ ‘stats’
      All declared Imports should be used.
    ```

# future.BatchJobs

Version: 0.15.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Henrik Bengtsson <henrikb@braju.com>’
    
    Insufficient package version (submitted: 0.15.0, existing: 0.15.0)
    
    This build time stamp is over a month old.
    ```

# gdalUtils

Version: 2.0.1.7

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Jonathan Asher Greenberg <gdalUtils@estarcion.net>’
    
    Insufficient package version (submitted: 2.0.1.7, existing: 2.0.1.7)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# gifti

Version: 0.7.4

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘John Muschelli <muschellij2@gmail.com>’
    
    Insufficient package version (submitted: 0.7.4, existing: 0.7.4)
    
    This build time stamp is over a month old.
    ```

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://img.shields.io/coveralls/muschellij2/gifti.svg
    TlsExceptionHostPort (HandshakeFailed Error_EOF) "img.shields.io" 443
    ```

# gofCopula

Version: 0.2-3

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Simon Trimborn <simon.trimborn@wiwi.hu-berlin.de>’
    
    Insufficient package version (submitted: 0.2.3, existing: 0.2.3)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# GSODR

Version: 1.0.7

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Adam Sparks <adamhsparks@gmail.com>’
    
    Insufficient package version (submitted: 1.0.7, existing: 1.0.7)
    
    Days since last update: 2
    ```

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://codecov.io/gh/ropensci/GSODR/branch/master/graph/badge.svg
    TlsExceptionHostPort (HandshakeFailed (Error_Packet_unexpected "Alert [(AlertLevel_Fatal,BadRecordMac)]" " expected: change cipher")) "codecov.io" 443
    ```

# hashFunction

Version: 1.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Xiaowei Zhan <zhanxw@gmail.com>’
    
    Insufficient package version (submitted: 1.0, existing: 1.0)
    
    The Title field should be in title case, current version then in title case:
    ‘A collection of non-cryptographic hash functions’
    ‘A Collection of Non-Cryptographic Hash Functions’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking for code which exercises the package ... WARNING
    ```
    No examples, no tests, no vignettes
    ```

*   checking package namespace information ... NOTE
    ```
      Namespace with empty importFrom: ‘utils’
    ```

*   checking top-level files ... NOTE
    ```
    Non-standard file/directory found at top level:
      ‘test’
    ```

# hoardeR

Version: 0.9.2

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Daniel Fischer <daniel.fischer@luke.fi>’
    
    Insufficient package version (submitted: 0.9.2, existing: 0.9.2)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# HyPhy

Version: 1.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Nathaniel Malachi Hallinan <nmhallinan@gmail.com>’
    
    Insufficient package version (submitted: 1.0, existing: 1.0)
    
    The Title field should be in title case, current version then in title case:
    ‘Macroevolutionary phylogentic analysis of species trees and gene trees’
    ‘Macroevolutionary Phylogentic Analysis of Species Trees and Gene Trees’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Author field differs from that derived from Authors@R
      Author:    ‘Nathaniel Malachi Hallinan’
      Authors@R: ‘Nathaniel Hallinan [aut, cre]’
    
    Maintainer field differs from that derived from Authors@R
      Maintainer: ‘Nathaniel Malachi Hallinan <nmhallinan@gmail.com>’
      Authors@R:  ‘Nathaniel Hallinan <nmhallinan@gmail.com>’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘R.utils’ ‘ape’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    requiprobable: no visible global function definition for ‘extract’
    requiprobable : make.tree: no visible global function definition for
      ‘extract’
    requiprobable : make.tree: no visible global function definition for
      ‘runif’
    rgenetree: no visible global function definition for ‘dbinom’
    rgenetree: no visible global function definition for ‘dnbinom’
    rgenetree: no visible global function definition for ‘runif’
    rgenetree: no visible global function definition for ‘rbinom’
    rgenetree: no visible global function definition for ‘rnbinom’
    Undefined global functions or variables:
      dbinom dnbinom extract rbinom rnbinom runif
    Consider adding
      importFrom("stats", "dbinom", "dnbinom", "rbinom", "rnbinom", "runif")
    to your NAMESPACE file.
    ```

# joinXL

Version: 1.0.1

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Yvonne Glanville <yvonneglanville@gmail.com>’
    
    Insufficient package version (submitted: 1.0.1, existing: 1.0.1)
    
    This build time stamp is over a month old.
    ```

# listenv

Version: 0.6.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Henrik Bengtsson <henrikb@braju.com>’
    
    Insufficient package version (submitted: 0.6.0, existing: 0.6.0)
    
    This build time stamp is over a month old.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'listenv.md.rsp' failed with diagnostics:
    there is no package called ‘markdown’
    Execution halted
    ```

# LSPFP

Version: 1.0.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Rafael Dellen <Rafael.Dellen@uni-duesseldorf.de>’
    
    Insufficient package version (submitted: 1.0.0, existing: 1.0.0)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘tcltk2’
      All declared Imports should be used.
    ```

# mailR

Version: 0.4.1

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Rahul Premraj <r.premraj+mailR@gmail.com>’
    
    Insufficient package version (submitted: 0.4.1, existing: 0.4.1)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# mcparallelDo

Version: 1.1.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Russell S. Pierce <russell.s.pierce@gmail.com>’
    
    Insufficient package version (submitted: 1.1.0, existing: 1.1.0)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch http://img.shields.io/badge/license-GPL%20%28%3E=%202%29-brightgreen.svg?style=flat
    TlsExceptionHostPort (HandshakeFailed Error_EOF) "img.shields.io" 80
    ```

# methylKit

Version: 1.2.4

## In both

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://dl.dropboxusercontent.com/u/1373164/methylKit_logo.png
    StatusCodeException (Status {statusCode = 404, statusMessage = "Not Found"}) [("Server","nginx"),("Date","Thu, 19 Oct 2017 16:25:39 GMT"),("Content-Type","text/html"),("Transfer-Encoding","chunked"),("Connection","keep-alive"),("Vary","Accept-Encoding"),("X-Dropbox-Request-Id","869dc4764f4d00bfded6f7c4b73b21d5"),("X-Robots-Tag","noindex, nofollow, noimageindex"),("Content-Encoding","gzip"),("X-Response-Body-Start","<!DOCTYPE html>\n<html>\n<head><meta http-equiv=\"Content-Type\" content=\"text/html; charset=utf-8\">\n<title>Dropbox - 404</title>\n<link href=\"https://cfl.dropboxstatic.com/static/css/error.css\" rel=\"stylesheet\" type=\"text/css\"/>\n<link rel=\"shortcut icon\" href=\"https://cfl.dropboxstatic.com/static/images/favicon.ico\"/>\n\n</head>\n<body>\n<div class=\"figure\">\n<img src=\"https://cfl.dropboxstatic.com/static/images/publicfoldersunset.svg\" alt=\"Error: 404\"/>\n</div>\n<div id=\"errorbox\">\n<div class=\"public-folder-404\"> <h1 class=\"error_message\">File not found</h1> <br> <div class=\"message_body\"> Sorry, that file doesn&rsquo;t live here anymore. It might have been moved or made private. </div> <br> <a class=\"c-btn c-btn--primary public-folder-link\" href=\"https://www.dropbox.com\">Learn about Dropbox</a> </div>\n</div>\n<section class=\"footer-row\"><footer id=\"homepage-footer\" class=\"twelve-column dropbox-footer clearfix\"><nav><ul class=\"o-nav\"><li class=\"o-nav__header\">Dropbox</li><li class=\"o-nav__item\"><a href=\"https://www.dropbox.com/install\">Desktop app</a></li><li class=\"o-nav__item\"><a href=\"https://www.dropbox.com/mobile\">Mobile apps</a></li><li class=\"o-nav__item\"><a href=\"https://www.dropbox.com/plans?trigger=homepagefoot\">Plans</a></li><li class=\"o-nav__item\"><a href=\"https://www.dropbox.com/security\">Security</a></li></ul><ul class=\"o-nav\"><li class=\"o-nav__header\">Company</li><li class=\"o-nav__item\"><a href=\"https://www.dropbox.com/about\">About us</a></li><li class=\"o-nav__item\"><a href=\"https://bit.ly/dbxdotcomapps\">Jobs</a></li><li class=\"o-nav__item\"><a href=\"https://www.dropbox.com/news\">Press</a></li><li class=\"o-nav__item\"><a href=\"https://blogs.dropbox.com/dropbox\">Blog</a></li></ul><ul class=\"o-nav\"><li class=\"o-nav__header\">Support</li><li class=\"o-nav__item\"><a href=\"https://www.dropbox.com/help\">Help Center</a></li><li class=\"o-nav__item\"><a href=\"https://www.dropbox.com/contact\">Contact us</a></li><li class=\"o-nav__item\"><a href=\"https://www.dropbox.com/terms/cookies\">Cookies</a></li><li class=\"o-nav__item\"><a href=\"https://www.dropbox.com/privacy\">Privacy &amp; terms</a></li></ul><ul class=\"o-nav\"><li class=\"o-nav__header\">Community</li><li class=\"o-nav__item\"><a href=\"https://www.dropbox.com/developers\">Developers</a></li><li class=\"o-nav__item\"><a href=\"https://www.dropbox.com/referrals\">Referrals</a></li><li class=\"o-nav__item\"><a href=\"https://www.dropboxforum.com\">Forum</a></li></ul><ul class=\"o-nav\"><li class=\"o-nav__header\">Connect</li><li class=\"o-nav__item\"><a href=\"https://twitter.com/dropbox\">Twitter</a></li><li class=\"o-nav__item\"><a href=\"https://www.facebook.com/Dropbox\">Facebook</a></li><li class=\"o-nav__item\"><a href=\"https://plus.google.com/+Dropbox/posts\">Google+</a></li><li class=\"o-nav__item\"><a href=\"https://www.youtube.com/user/dropbox\">YouTube</a></li></ul><ul class=\"o-nav\"><li class=\"o-nav__header\">Products</li><li class=\"o-nav__item\"><a href=\"https://www.dropbox.com/pro\">Plus</a></li><li class=\"o-nav__item\"><a href=\"https://www.dropbox.com/business\">Business</a></li><li class=\"o-nav__item\"><a href=\"https://www.dropbox.com/enterprise\">Enterprise</a></li></ul></nav><div class=\"react-locale-selector-wrapper\"><div id=\"component1937679262106323703\"><div class=\"react-stub\"></div></div></div></footer></section>\n</body>\n</html>\n"),("X-Request-URL","GET https://dl.dropboxusercontent.com:443/u/1373164/methylKit_logo.png")] (CJ {expose = []})
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Altuna Akalin <aakalin@gmail.com>’
    
    Package duplicated from https://bioconductor.org/packages/3.5/bioc
    
    The Title field should be in title case, current version then in title case:
    ‘DNA methylation analysis from high-throughput bisulfite sequencing results’
    ‘DNA Methylation Analysis from High-Throughput Bisulfite Sequencing Results’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    This build time stamp is over a month old.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        doc    1.6Mb
        libs   1.7Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘KernSmooth’
      All declared Imports should be used.
    ```

# MPAgenomics

Version: 1.1.2

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Samuel Blanck <samuel.blanck@inria.fr>’
    
    Insufficient package version (submitted: 1.1.2, existing: 1.1.2)
    
    No package encoding and non-ASCII characters in the following R files:
      R/bivariateSignal.R
        64: # etrange=c(2,36)#66 semble greff<c3><a9> #7 CN <c3><a9>trange
      R/cghcall_private.R
        140:     #TODO v<c3><a9>rifier pertinence correction bug avec segment extreme d'un seul point
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘R.devices’ ‘R.filesets’ ‘R.methodsS3’ ‘R.oo’ ‘aroma.affymetrix’
      ‘aroma.cn’ ‘aroma.core’ ‘aroma.light’ ‘matrixStats’ ‘snowfall’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Unexported object imported by a ':::' call: ‘cghseg:::segmeanCO’
      See the note in ?`:::` about the use of this operator.
    ```

# msgbsR

Version: 1.0.0

## In both

*   checking data for ASCII and uncompressed saves ... WARNING
    ```
      
      Note: significantly better compression could be obtained
            by using R CMD build --resave-data
                   old_size new_size compress
      ratdata.rda     318Kb    128Kb       xz
      ratdata2.rda    287Kb    116Kb       xz
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Benjamin Mayne <benjamin.mayne@adelaide.edu.au>’
    
    Package duplicated from https://bioconductor.org/packages/3.5/bioc
    
    The Title field starts with the package name.
    The Title field should be in title case, current version then in title case:
    ‘msgbsR: methylation sensitive genotyping by sequencing (MS-GBS) R functions’
    ‘msgbsR: Methylation Sensitive Genotyping by Sequencing (MS-GBS) R Functions’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    
    Size of tarball: 16297733 bytes
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 16.1Mb
      sub-directories of 1Mb or more:
        extdata  15.2Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: 'utils'
      All declared Imports should be used.
    ```

# MSIseq

Version: 1.0.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    ...
    
    Insufficient package version (submitted: 1.0.0, existing: 1.0.0)
    
    Found the following (possibly) invalid URLs:
      URL: https://tcga-data.nci.nih.gov/tcga/
        From: man/NGStestdata.Rd
              man/NGStestseqLen.Rd
              man/NGStesttype.Rd
              man/NGStrainclass.Rd
              man/NGStraindata.Rd
              man/NGStrainseqLen.Rd
              man/NGStraintype.Rd
              man/test.mutationNum.Rd
              man/train.mutationNum.Rd
        Status: Error
        Message: libcurl error code 47:
        	Maximum (20) redirects followed
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# nandb

Version: 0.2.1

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Rory Nolan <rorynoolan@gmail.com>’
    
    Insufficient package version (submitted: 0.2.1, existing: 0.2.1)
    
    This build time stamp is over a month old.
    ```

# net.security

Version: 0.3.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Humbert Costas <humbert.costas@gmail.com>’
    
    Insufficient package version (submitted: 0.3.0, existing: 0.3.0)
    
    This build time stamp is over a month old.
    ```

# neurobase

Version: 1.13.2

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘John Muschelli <muschellij2@gmail.com>’
    
    Insufficient package version (submitted: 1.13.2, existing: 1.13.2)
    
    This build time stamp is over a month old.
    ```

# NSA

Version: 0.0.32

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Maria Ortiz-Estevez <mortizest@gmail.com>’
    
    Insufficient package version (submitted: 0.0.32, existing: 0.0.32)
    
    The Title field should be in title case, current version then in title case:
    ‘Post-normalization of total copy numbers’
    ‘Post-Normalization of Total Copy Numbers’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘R.methodsS3’ ‘MASS’ ‘matrixStats’ ‘R.oo’ ‘R.utils’ ‘aroma.core’
      ‘aroma.affymetrix’ ‘DNAcopy’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking top-level files ... NOTE
    ```
    Non-standard file/directory found at top level:
      ‘incl’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘DNAcopy’ ‘MASS’ ‘R.methodsS3’ ‘R.oo’ ‘aroma.affymetrix’ ‘aroma.core’
      ‘matrixStats’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking S3 generic/method consistency ... NOTE
    ```
    Found the following apparent S3 methods exported but not registered:
      NSAByTotalAndFracB.matrix allocateOutputDataSets.NSANormalization
      allocateOutputDataSets.SNPsNormalization
      allocateOutputDataSets.SampleNormalization
      as.character.NSANormalization as.character.SNPsNormalization
      as.character.SampleNormalization findArraysTodo.NSANormalization
      findArraysTodo.SampleNormalization findUnitsTodo.SNPsNormalization
      fitNSA.matrix fitNSAcnPs.matrix getDataSets.NSANormalization
      getDataSets.SNPsNormalization getDataSets.SampleNormalization
      getName.NSANormalization getName.SNPsNormalization
      getName.SampleNormalization getOutputDataSets.NSANormalization
      getOutputDataSets.SNPsNormalization
      getOutputDataSets.SampleNormalization getPath.NSANormalization
      getPath.SNPsNormalization getPath.SampleNormalization
      getRootPath.NSANormalization getRootPath.SNPsNormalization
      getRootPath.SampleNormalization process.NSANormalization
      process.SNPsNormalization process.SampleNormalization
      sampleNByTotalAndFracB.numeric snpsNByTotalAndFracB.matrix
    See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
    manual.
    ```

# oce

Version: 0.9-22

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Dan Kelley <Dan.Kelley@Dal.Ca>’
    
    Insufficient package version (submitted: 0.9.22, existing: 0.9.22)
    
    This build time stamp is over a month old.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        help   2.2Mb
    ```

# officer

Version: 0.1.8

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘officer-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: slip_in_img
    > ### Title: append an image
    > ### Aliases: slip_in_img
    > 
    > ### ** Examples
    > 
    > library(magrittr)
    > img.file <- file.path( Sys.getenv("R_HOME"), "doc", "html", "logo.jpg" )
    > x <- read_docx() %>%
    +   body_add_par("R logo: ", style = "Normal") %>%
    +   slip_in_img(src = img.file, style = "strong", width = .3, height = .3)
    Error: file.exists(src) is not TRUE
    Execution halted
    ```

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘David Gohel <david.gohel@ardata.fr>’
    
    Insufficient package version (submitted: 0.1.8, existing: 0.1.8)
    
    Number of updates in past 6 months: 7
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    
    Attaching package: 'dplyr'
    
    The following objects are masked from 'package:stats':
    
        filter, lag
    
    The following objects are masked from 'package:base':
    
        intersect, setdiff, setequal, union
    
    Quitting from lines 180-190 (powerpoint.Rmd) 
    Error: processing vignette 'powerpoint.Rmd' failed with diagnostics:
    file.exists(src) is not TRUE
    Execution halted
    ```

# oneChannelGUI

Version: 1.42.0

## In both

*   checking sizes of PDF files under ‘inst/doc’ ... WARNING
    ```
      ‘gs+qpdf’ made some significant size reductions:
         compacted ‘Exon-level.analysis.pdf’ from 1395Kb to 598Kb
         compacted ‘RNAseq.pdf’ from 1979Kb to 385Kb
      consider running tools::compactPDF(gs_quality = "ebook") on these files
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    ...
      R/exonmenu.R
        29: #              .cosie adapted from the article of Gaidatzis Nucleic Acids Research, 2009, 1<96>10
      R/qcmenu.R
        369:                #######################################################<e0><e0>
        405:                #######################################################<e0><e0>
        406:                #######################################################<e0><e0>
        442:                #######################################################<e0><e0>
        660:                       #######################################################<e0><e0>
    
    The Title field should be in title case, current version then in title case:
    ‘A graphical interface designed to facilitate analysis of microarrays and miRNA/RNA-seq data on laptops’
    ‘A Graphical Interface Designed to Facilitate Analysis of Microarrays and miRNA/RNA-Seq Data on Laptops’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is not in ISO 8601 yyyy-mm-dd format.
    
    This build time stamp is over a month old.
    
    Size of tarball: 11484094 bytes
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘Biobase’ ‘affylmGUI’ ‘tkrplot’ ‘tkWidgets’ ‘IRanges’ ‘Rsamtools’
      ‘Biostrings’ ‘siggenes’ ‘chimera’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        doc   5.0Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘BSgenome.Hsapiens.UCSC.hg19’ ‘BSgenome.Mmusculus.UCSC.mm9’
      ‘BSgenome.Rnorvegicus.UCSC.rn4’ ‘Genominator’ ‘affy’ ‘affyPLM’
      ‘chipseq’ ‘maSigPro’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

# openCyto

Version: 1.14.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in engine$weave(file, quiet = quiet, encoding = enc) :
      The vignette engine knitr::rmarkdown is not available, because the rmarkdown package is not installed. Please install it.
    Loading required package: flowWorkspace
    Loading required package: flowCore
    Loading required package: ncdfFlow
    Loading required package: RcppArmadillo
    Loading required package: BH
    Quitting from lines 30-34 (HowToAutoGating.Rmd) 
    Error: processing vignette 'HowToAutoGating.Rmd' failed with diagnostics:
    there is no package called 'ggcyto'
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Mike Jiang <wjiang2@fhcrc.org>’
    
    Package duplicated from https://bioconductor.org/packages/3.5/bioc
    
    The Title field should be in title case, current version then in title case:
    ‘Hierarchical Gating Pipeline for flow cytometry data’
    ‘Hierarchical Gating Pipeline for Flow Cytometry Data’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘flowWorkspace’
    A package should be listed in only one of these fields.
    ```

*   checking top-level files ... NOTE
    ```
    Non-standard file/directory found at top level:
      ‘example’
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘parallel’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Namespace in Imports field not imported from: ‘flowViz’
      All declared Imports should be used.
    Unexported objects imported by ':::' calls:
      ‘flowClust:::.ellipsePoints’ ‘flowStats:::drvkde’
      ‘flowStats:::warpSetNCDF’ ‘flowWorkspace:::.cpp_addGate’
      ‘flowWorkspace:::.getAllDescendants’ ‘flowWorkspace:::trimWhiteSpace’
      ‘lattice:::updateList’
      See the note in ?`:::` about the use of this operator.
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      ‘groupBy’ ‘isCollapse’ ‘ppMethod’ ‘unlockNamespace’
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' call not declared from: ‘ggcyto’
    ```

# PGA

Version: 1.6.0

## Newly fixed

*   R CMD check timed out
    

## In both

*   checking CRAN incoming feasibility ... ERROR
    ```
    Maintainer: ‘Bo Wen <wenbo@genomics.cn>, Shaohang Xu <xsh.skye@gmail.com>’
    
    The maintainer field is invalid or specifies more than one person
    
    Conflicting package names (submitted: PGA, existing: pga [CRAN archive])
    
    Package duplicated from https://bioconductor.org/packages/3.5/bioc
    
    Package has FOSS license, installs .class/.jar but has no 'java' directory.
    
    The Title field should be in title case, current version then in title case:
    ‘An package for identification of novel peptides by customized database derived from RNA-Seq’
    ‘An Package for Identification of Novel Peptides by Customized Database Derived from RNA-Seq’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        extdata   1.8Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘biomaRt:::martBM’ ‘biomaRt:::martDataset’ ‘biomaRt:::martHost’
      ‘customProDB:::makeTranscriptDbFromBiomart_archive’
      See the note in ?`:::` about the use of this operator.
    ```

# PhenotypeSimulator

Version: 0.1.3

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Hannah Meyer <hannah@ebi.ac.uk>’
    
    Insufficient package version (submitted: 0.1.3, existing: 0.1.3)
    
    This build time stamp is over a month old.
    ```

# PopGenReport

Version: 3.0.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Bernd Gruber <bernd.gruber@canberra.edu.au>’
    
    Insufficient package version (submitted: 3.0.0, existing: 3.0.0)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘ecodist’
    ```

# proteoQC

Version: 1.12.3

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'proteoQC.Rmd' failed with diagnostics:
    there is no package called ‘prettydoc’
    Execution halted
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    ...
    
    Package has FOSS license, installs .class/.jar but has no 'java' directory.
    
    Found the following (possibly) invalid URLs:
      URL: http://CRAN.R-project.org/package=Nozzle.R1
        From: inst/doc/proteoQC.html
        Status: 200
        Message: OK
        CRAN URL not in canonical form
      Canonical CRAN.R-project.org URLs use https.
    
    The Title field should be in title case, current version then in title case:
    ‘An R package for proteomics data quality control’
    ‘An R Package for Proteomics Data Quality Control’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    This build time stamp is over a month old.
    
    Size of tarball: 5837430 bytes
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.0Mb
      sub-directories of 1Mb or more:
        doc       3.2Mb
        extdata   4.0Mb
    ```

*   checking S3 generic/method consistency ... NOTE
    ```
    Found the following apparent S3 methods exported but not registered:
      print.msQCres
    See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
    manual.
    ```

# psichomics

Version: 1.2.1

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Nuno Saraiva-Agostinho <nunodanielagostinho@gmail.com>’
    
    Package duplicated from https://bioconductor.org/packages/3.5/bioc
    
    This build time stamp is over a month old.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Authors@R field gives persons with non-standard roles:
      Nuno Barbosa-Morais [aut, led, ths]: led
    ```

# PubBias

Version: 1.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Simon Thornley <sithor@gmail.com>’
    
    Insufficient package version (submitted: 1.0, existing: 1.0)
    
    The Title field should be in title case, current version then in title case:
    ‘Performs simulation study to look for publication bias, using a technique described by Ioannidis and Trikalinos; Clin Trials. 2007;4(3):245-53.’
    ‘Performs Simulation Study to Look for Publication Bias, using a Technique Described by Ioannidis and Trikalinos; Clin Trials. 2007;4(3):245-53.’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘R.utils’ ‘rmeta’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ChisqTest_expect: no visible global function definition for ‘pchisq’
    plot_chase_observed_expected: no visible global function definition for
      ‘meta.MH’
    plot_chase_observed_expected: no visible global function definition for
      ‘txtProgressBar’
    plot_chase_observed_expected: no visible global function definition for
      ‘setTxtProgressBar’
    test.one.treated: no visible global function definition for
      ‘fisher.test’
    test_one: no visible global function definition for ‘rbinom’
    test_one: no visible global function definition for ‘fisher.test’
    Undefined global functions or variables:
      fisher.test meta.MH pchisq rbinom setTxtProgressBar txtProgressBar
    Consider adding
      importFrom("stats", "fisher.test", "pchisq", "rbinom")
      importFrom("utils", "setTxtProgressBar", "txtProgressBar")
    to your NAMESPACE file.
    ```

# QDNAseq

Version: 1.12.0

## In both

*   checking for missing documentation entries ... WARNING
    ```
    Undocumented code objects:
      ‘exportVCF’
    All user-level objects in a package should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Daoud Sie <d.sie@vumc.nl>’
    
    Package duplicated from https://bioconductor.org/packages/3.5/bioc
    
    Package has a FOSS license but eventually depends on the following
    package which may restrict use:
      CGHcall
    
    The Title field should be in title case, current version then in title case:
    ‘Quantitative DNA sequencing for chromosomal aberrations’
    ‘Quantitative DNA Sequencing for Chromosomal Aberrations’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# qrcode

Version: 0.1.1

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Victor Teh <victorteh@gmail.com>’
    
    Insufficient package version (submitted: 0.1.1, existing: 0.1.1)
    
    The Title field starts with the package name.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# R.cache

Version: 0.12.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Henrik Bengtsson <henrikb@braju.com>’
    
    Insufficient package version (submitted: 0.12.0, existing: 0.12.0)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# R.devices

Version: 2.15.1

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Henrik Bengtsson <henrikb@braju.com>’
    
    Insufficient package version (submitted: 2.15.1, existing: 2.15.1)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# R.filesets

Version: 2.11.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Henrik Bengtsson <henrikb@braju.com>’
    
    Insufficient package version (submitted: 2.11.0, existing: 2.11.0)
    
    This build time stamp is over a month old.
    ```

# R.huge

Version: 0.9.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Henrik Bengtsson <henrikb@braju.com>’
    
    Insufficient package version (submitted: 0.9.0, existing: 0.9.0)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# R.matlab

Version: 3.6.1

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Henrik Bengtsson <henrikb@braju.com>’
    
    Insufficient package version (submitted: 3.6.1, existing: 3.6.1)
    
    Found the following (possibly) invalid URLs:
      URL: http://www.mathworks.com/matlabcentral/fileexchange/loadFile.do?objectId=5051
        From: man/R.matlab-package.Rd
        Status: Error
        Message: libcurl error code 52:
        	Empty reply from server
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# R.rsp

Version: 0.41.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Henrik Bengtsson <henrikb@braju.com>’
    
    Insufficient package version (submitted: 0.41.0, existing: 0.41.0)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# ReporteRs

Version: 0.8.8

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘David Gohel <david.gohel@lysis-consultants.fr>’
    
    Insufficient package version (submitted: 0.8.8, existing: 0.8.8)
    
    This build time stamp is over a month old.
    ```

# ReportingTools

Version: 2.16.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Jason A. Hackney <hackney.jason@gene.com>, Gabriel Becker <becker.gabe@gene.com>, Jessica L. Larson <larson.jessica@gmail.com>’
    
    The maintainer field is invalid or specifies more than one person
    
    Package duplicated from https://bioconductor.org/packages/3.5/bioc
    
    The Title field should be in title case, current version then in title case:
    ‘Tools for making reports in various formats’
    ‘Tools for Making Reports in Various Formats’
    
    This build time stamp is over a month old.
    ```

# RforProteomics

Version: 1.14.0

## In both

*   checking sizes of PDF files under ‘inst/doc’ ... WARNING
    ```
      ‘gs+qpdf’ made some significant size reductions:
         compacted ‘HUPO2014poster.pdf’ from 1127Kb to 791Kb
      consider running tools::compactPDF(gs_quality = "ebook") on these files
    ```

*   checking CRAN incoming feasibility ... NOTE
    ```
    ...
        CRAN URL not in canonical form
      URL: http://sebastiangibb.de/research.html
        From: inst/doc/RProtVis.html
        Status: Error
        Message: libcurl error code 6:
        	Could not resolve host: sgibb.github.ioresearch.html
      The canonical URL of the CRAN page for a package is 
        https://CRAN.R-project.org/package=pkgname
      The canonical URL of the CRAN page for a task view is 
        https://CRAN.R-project.org/view=viewname
    
    The Title field should be in title case, current version then in title case:
    ‘Companion package to the 'Using R and Bioconductor for proteomics data analysis' publication’
    ‘Companion Package to the 'Using R and Bioconductor for proteomics data analysis' Publication’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    This build time stamp is over a month old.
    
    Size of tarball: 8490435 bytes
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking: ‘MSGFplus’ ‘MSGFgui’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.0Mb
      sub-directories of 1Mb or more:
        doc   7.1Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Author field differs from that derived from Authors@R
      Author:    ‘Laurent Gatto [aut, cre], Thomas Lin Pedersen [ctb], Sebastian Gibb [ctb], Vlad Petyuk [ctb]’
      Authors@R: ‘Laurent Gatto [aut, cre], Sebastian Gibb [ctb], Vlad Petyuk [ctb], Thomas Pedersen Lin [ctb]’
    ```

*   checking top-level files ... NOTE
    ```
    Non-standard file/directory found at top level:
      ‘external_data_store.txt’
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' import not declared from: ‘AnnotationDbi’
    ```

# Rnightlights

Version: 0.1.2

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Christopher Njuguna <chris.njuguna@gmail.com>’
    
    Insufficient package version (submitted: 0.1.2, existing: 0.1.2)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# ROI.models.miplib

Version: 0.0-1

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Florian Schwendinger <FlorianSchwendinger@gmx.at>’
    
    Insufficient package version (submitted: 0.0.1, existing: 0.0.1)
    
    This build time stamp is over a month old.
    ```

# RxODE

Version: 0.6-1

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Wenping Wang <wwang8198@gmail.com>’
    
    Insufficient package version (submitted: 0.6.1, existing: 0.6.1)
    
    Days since last update: 6
    ```

# sdmpredictors

Version: 0.2.6

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Samuel Bosch <mail@samuelbosch.com>’
    
    Insufficient package version (submitted: 0.2.6, existing: 0.2.6)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://img.shields.io/github/license/samuelbosch/blogbits.svg
    TlsExceptionHostPort (HandshakeFailed Error_EOF) "img.shields.io" 443
    ```

# SDMTools

Version: 1.1-221

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    ...
      URL: http://williams.best.vwh.net/avform.htm#Crs
        From: man/distance.Rd
        Status: Error
        Message: libcurl error code 6:
        	Could not resolve host: williams.best.vwh.net
      URL: http://www.gnu.org/software/gsl/manual/html_node/Weighted-Samples.htmland
        From: man/wt.mean.Rd
        Status: 404
        Message: Not Found
      Spaces in an http[s] URL should probably be replaced by %20
    
    The Title field should be in title case, current version then in title case:
    ‘Species Distribution Modelling Tools: Tools for processing data associated with species distribution modelling exercises’
    ‘Species Distribution Modelling Tools: Tools for Processing Data Associated with Species Distribution Modelling Exercises’
    
    The Description field should not start with the package name,
      'This package' or similar.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘raster’ ‘sp’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    quick.map: no visible global function definition for ‘image’
    raster.from.asc: no visible global function definition for ‘raster’
    raster.from.asc: no visible global function definition for
      ‘projection<-’
    raster.from.asc: no visible global function definition for ‘setValues’
    slope: no visible global function definition for ‘setValues’
    sp.from.asc: no visible global function definition for ‘CRS’
    sp.from.asc: no visible global function definition for ‘GridTopology’
    sp.from.asc: no visible global function definition for
      ‘SpatialGridDataFrame’
    Undefined global functions or variables:
      CRS GridTopology SpatialGridDataFrame aggregate contour getValues
      gray heat.colors image na.omit pnorm polygon projection<- quantile
      rainbow raster rect sd segments setValues terrain.colors text
    Consider adding
      importFrom("grDevices", "gray", "heat.colors", "rainbow",
                 "terrain.colors")
      importFrom("graphics", "contour", "image", "polygon", "rect",
                 "segments", "text")
      importFrom("stats", "aggregate", "na.omit", "pnorm", "quantile", "sd")
    to your NAMESPACE file.
    ```

# SEMID

Version: 0.3.1

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Luca Weihs <lucaw@uw.edu>’
    
    Insufficient package version (submitted: 0.3.1, existing: 0.3.1)
    
    Days since last update: 5
    ```

# SensusR

Version: 2.2.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Matthew S. Gerber <gerber.matthew@gmail.com>’
    
    Insufficient package version (submitted: 2.2.0, existing: 2.2.0)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# simPATHy

Version: 0.2

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Elisa Salviato <elisa.salviato.88@gmail.com>’
    
    Insufficient package version (submitted: 0.2, existing: 0.2)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# snplist

Version: 0.18

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Alexander Sibley <alexander.sibley@dm.duke.edu>’
    
    Insufficient package version (submitted: 0.18, existing: 0.18)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# SpaDES.core

Version: 0.1.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Alex M Chubaty <alexander.chubaty@canada.ca>’
    
    Insufficient package version (submitted: 0.1.0, existing: 0.1.0)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘CircStats’ ‘RandomFields’ ‘grDevices’ ‘sp’
      All declared Imports should be used.
    ```

# spm12r

Version: 2.3.1

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘John Muschelli <muschellij2@gmail.com>’
    
    Insufficient package version (submitted: 2.3.1, existing: 2.3.1)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘utils’
      All declared Imports should be used.
    ```

# stplanr

Version: 0.1.9

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Robin Lovelace <rob00x@gmail.com>’
    
    Insufficient package version (submitted: 0.1.9, existing: 0.1.9)
    
    This build time stamp is over a month old.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘tmap’
    ```

# taxonomizr

Version: 0.2.2

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Scott Sherrill-Mix <shescott@upenn.edu>’
    
    Insufficient package version (submitted: 0.2.2, existing: 0.2.2)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# TBSSurvival

Version: 1.3

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Cassio de Campos <cassiopc@acm.org>’
    
    Insufficient package version (submitted: 1.3, existing: 1.3)
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# TCGAbiolinks

Version: 2.5.9

## In both

*   R CMD check timed out
    

*   checking CRAN incoming feasibility ... WARNING
    ```
    ...
        Message: Invalid URI scheme
      URL: GO:cellular
        From: inst/doc/tcgaBiolinks.html
        Message: Invalid URI scheme
      URL: GO:molecular
        From: inst/doc/tcgaBiolinks.html
        Message: Invalid URI scheme
      URL: doi:10.1016/j.ccell.2016.04.002
        From: inst/doc/tcgaBiolinks.html
        Message: Invalid URI scheme (use \doi for DOIs in Rd markup)
    
    The Title field starts with the package name.
    The Title field should be in title case, current version then in title case:
    ‘TCGAbiolinks: An R/Bioconductor package for integrative analysis with GDC data’
    ‘TCGAbiolinks: An R/Bioconductor Package for Integrative Analysis with GDC Data’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    
    Size of tarball: 44816700 bytes
    ```

*   checking top-level files ... WARNING
    ```
    Conversion of ‘README.md’ failed:
    pandoc: Could not fetch https://img.shields.io/badge/doi-10.1093/nar/gkv1507-green.svg?style=flat
    TlsExceptionHostPort (HandshakeFailed Error_EOF) "img.shields.io" 443
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 61.0Mb
      sub-directories of 1Mb or more:
        R      1.1Mb
        data   2.3Mb
        doc   57.4Mb
    ```

# tmle.npvi

Version: 0.10.0

## In both

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Pierre Neuvial <pierre.neuvial@genopole.cnrs.fr>’
    
    Insufficient package version (submitted: 0.10.0, existing: 0.10.0)
    
    Found the following (possibly) invalid URLs:
      URL: http://CRAN.R-project.org/package=tmle.npvi
        From: inst/CITATION
        CRAN URL not in canonical form
      URL: http://supportres.illumina.com/documents/myillumina/b78d361a-def5-4adb-ab38-e8990625f053/humanmethylation450_15017482_v1-2.csv
        From: man/tcga2012brca.Rd
        Status: 403
        Message: Forbidden
      Canonical CRAN.R-project.org URLs use https.
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

# ToPASeq

Version: 1.10.0

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘Ivana Ihnatova <ihnatova@iba.muni.cz>’
    
    Package duplicated from https://bioconductor.org/packages/3.5/bioc
    
    The Title field should be in title case, current version then in title case:
    ‘Package for Topology-based Pathway Analysis of RNASeq data’
    ‘Package for Topology-Based Pathway Analysis of RNASeq Data’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    
    CRAN-pack does not know about
      .BBSoptions
    ```

*   checking top-level files ... NOTE
    ```
    Non-standard file/directory found at top level:
      ‘Rplots.pdf’
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘DEGraph’ ‘plotrix’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

# Uniquorn

Version: 1.4.2

## In both

*   checking CRAN incoming feasibility ... NOTE
    ```
    Maintainer: ‘'Raik Otto' <raik.otto@hu-berlin.de>’
    
    Package duplicated from https://bioconductor.org/packages/3.5/bioc
    
    The Title field should be in title case, current version then in title case:
    ‘Identification of cancer cell lines based on their weighted mutational or variational fingerprint’
    ‘Identification of Cancer Cell Lines Based on their Weighted Mutational or Variational Fingerprint’
    
    The Date field is over a month old.
    
    This build time stamp is over a month old.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Author field differs from that derived from Authors@R
      Author:    ‘Raik Otto’
      Authors@R: ‘Raik Otto [aut, cre]’
    
    Maintainer field differs from that derived from Authors@R
      Maintainer: ‘‘Raik Otto’ <raik.otto@hu-berlin.de>’
      Authors@R:  ‘Raik Otto <raik.otto@hu-berlin.de>’
    ```

*   checking top-level files ... NOTE
    ```
    Non-standard file/directory found at top level:
      ‘unitTests’
    ```

# vortexR

Version: 1.1.4

## In both

*   checking examples ... ERROR
    ```
    ...
    > ### Name: SSMD_matrix
    > ### Title: Generate a SSMD matrix with all possible pairwise comparisons
    > ### Aliases: SSMD_matrix
    > 
    > ### ** Examples
    > 
    > # Using Campbell et al. and Pacioni et al. example data.
    > # See ?pacioni and ?campbell for more details on example data.
    > require(vortexRdata)
    Loading required package: vortexRdata
    > 
    > data("pac.clas")
    > 
    > SSMD_matrix(data=pac.clas, project="Pacioni_et_al",
    + scenario="ST_Classic",
    + params = c("PExtinct", "Nextant", "Het", "Nalleles"),
    + yrs = c(60, 120), ST = FALSE, save2disk = FALSE)
    Error in if (!yrs == "max" & !is.numeric(yrs)) stop("invalid value(s) for 'yrs' ") : 
      the condition has length > 1
    Calls: SSMD_matrix
    Execution halted
    ```

*   checking CRAN incoming feasibility ... WARNING
    ```
    Maintainer: ‘Carlo Pacioni <C.Pacioni@Murdoch.edu.au>’
    
    Insufficient package version (submitted: 1.1.4, existing: 1.1.4)
    
    Found the following (possibly) invalid URLs:
      URL: https://doi.org/10.1071/PC17002
        From: man/pac.clas.Nadults.Rd
              man/pac.clas.Ne.Rd
              man/pac.clas.Rd
              man/pac.clas.lookup.Rd
              man/pac.clas.pairw.Rd
              man/pac.lhs.Rd
              man/pac.run.lhs.Rd
              man/pac.yr.Rd
              man/vortexR.Rd
        Status: 503
        Message: Service Unavailable
    
    This build time stamp is over a month old.
    ```

