# Setup

## Platform

|setting  |value                        |
|:--------|:----------------------------|
|version  |R version 3.3.2 (2016-10-31) |
|system   |x86_64, linux-gnu            |
|ui       |X11                          |
|language |en                           |
|collate  |en_US.UTF-8                  |
|tz       |US/Pacific                   |
|date     |2016-11-07                   |

## Packages

|package     |*  |version    |date       |source                             |
|:-----------|:--|:----------|:----------|:----------------------------------|
|digest      |   |0.6.10     |2016-08-02 |cran (@0.6.10)                     |
|R.methodsS3 |   |1.7.1      |2016-02-16 |CRAN (R 3.3.1)                     |
|R.oo        |   |1.21.0     |2016-11-01 |cran (@1.21.0)                     |
|R.utils     |   |2.4.0-9000 |2016-11-07 |local (HenrikBengtsson/R.utils@NA) |

# Check results

84 packages

|package          |version | errors| warnings| notes|
|:----------------|:-------|------:|--------:|-----:|
|ACNE             |0.8.1   |      0|        0|     0|
|acnr             |0.2.4   |      0|        0|     0|
|affxparser       |1.46.0  |      0|        1|     2|
|alphahull        |2.1     |      0|        0|     0|
|arc              |1.0     |      0|        0|     0|
|aroma.affymetrix |3.0.0   |      0|        0|     0|
|aroma.apd        |0.6.0   |      0|        0|     0|
|aroma.cn         |1.6.1   |      0|        0|     0|
|aroma.core       |3.0.0   |      0|        0|     1|
|aroma.light      |3.4.0   |      0|        0|     1|
|babel            |0.3-0   |      0|        0|     0|
|bedr             |1.0.3   |      0|        0|     0|
|bsseq            |1.10.0  |      0|        0|     0|
|calmate          |0.12.1  |      0|        0|     0|
|Causata          |4.2-0   |      0|        0|     5|
|ChemoSpec        |4.3.34  |      0|        0|     1|
|CNEr             |1.10.1  |      0|        2|     2|
|CollapsABEL      |0.10.8  |      0|        0|     0|
|DeepBlueR        |1.0.3   |      0|        0|     1|
|DEGraph          |1.26.0  |      0|        0|     6|
|demi             |1.1.2   |      0|        0|     1|
|downsize         |0.2.1   |      0|        0|     0|
|dplR             |1.6.4   |      0|        0|     0|
|DupChecker       |1.12.0  |      0|        0|     1|
|edgar            |1.0.6   |      0|        0|     0|
|eegc             |1.0.0   |      0|        0|     4|
|EEM              |1.1.1   |      0|        0|     0|
|eiCompare        |1.7     |      0|        0|     0|
|ELMER            |1.4.0   |      0|        0|     3|
|eudysbiome       |1.4.0   |      0|        0|     1|
|evclust          |1.0.3   |      0|        0|     0|
|ezknitr          |0.6     |      0|        0|     0|
|FGNet            |3.8.0   |      1|        1|     3|
|freesurfer       |1.0     |      0|        0|     0|
|fslr             |2.0     |      0|        0|     0|
|future.BatchJobs |0.13.1  |      0|        0|     0|
|gdalUtils        |2.0.1.7 |      0|        0|     0|
|gofCopula        |0.2-3   |      0|        0|     0|
|hashFunction     |1.0     |      0|        0|     2|
|hoardeR          |0.9.2   |      0|        0|     0|
|HyPhy            |1.0     |      0|        0|     3|
|joinXL           |1.0.1   |      0|        0|     0|
|listenv          |0.6.0   |      0|        0|     0|
|LSPFP            |1.0.0   |      0|        0|     0|
|mailR            |0.4.1   |      0|        0|     0|
|mcparallelDo     |1.1.0   |      0|        0|     0|
|methylKit        |1.0.0   |      0|        0|     0|
|MPAgenomics      |1.1.2   |      0|        0|     2|
|MSIseq           |1.0.0   |      0|        0|     1|
|neurobase        |1.1.1.1 |      0|        0|     0|
|NSA              |0.0.32  |      0|        0|     6|
|oce              |0.9-19  |      0|        0|     1|
|oneChannelGUI    |1.40.0  |      0|        2|     6|
|openCyto         |1.12.1  |      0|        2|     5|
|paxtoolsr        |1.8.0   |      0|        0|     2|
|PGA              |1.4.0   |      0|        0|     3|
|PopGenReport     |2.2.2   |      0|        0|     1|
|proteoQC         |1.10.0  |      0|        0|     4|
|PSCBS            |0.61.0  |      0|        0|     0|
|psichomics       |1.0.0   |      0|        1|     1|
|PubBias          |1.0     |      0|        0|     3|
|QDNAseq          |1.10.0  |      0|        0|     0|
|qrcode           |0.1.1   |      0|        0|     0|
|R.cache          |0.12.0  |      0|        0|     0|
|R.devices        |2.14.0  |      0|        0|     0|
|ReporteRs        |0.8.7   |      0|        0|     0|
|ReportingTools   |2.14.0  |      0|        0|     1|
|R.filesets       |2.10.0  |      0|        0|     0|
|R.huge           |0.9.0   |      0|        0|     0|
|R.matlab         |3.6.1   |      0|        0|     0|
|R.rsp            |0.30.0  |      0|        0|     0|
|rvg              |0.1.1   |      0|        0|     1|
|scphaser         |1.0.0   |      0|        0|     0|
|SDMTools         |1.1-221 |      0|        0|     2|
|simPATHy         |0.2     |      0|        0|     0|
|snplist          |0.16    |      0|        0|     0|
|SpaDES           |1.3.1   |      0|        2|     1|
|spm12r           |2.0     |      0|        0|     0|
|stplanr          |0.1.5   |      0|        1|     0|
|TBSSurvival      |1.2     |      0|        0|     1|
|TCGAbiolinks     |2.2.3   |      0|        0|     3|
|tmle.npvi        |0.10.0  |      0|        0|     1|
|ToPASeq          |1.8.0   |      0|        0|     5|
|Uniquorn         |1.2.0   |      0|        0|     2|

## ACNE (0.8.1)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/ACNE/issues

0 errors | 0 warnings | 0 notes

## acnr (0.2.4)
Maintainer: Morgane Pierre-Jean <morgane.pierrejean@genopole.cnrs.fr>

0 errors | 0 warnings | 0 notes

## affxparser (1.46.0)
Maintainer: Kasper Daniel Hansen <kasperdanielhansen@gmail.com>  
Bug reports: https://github.com/HenrikBengtsson/affxparser/issues

0 errors | 1 warning  | 2 notes

```
checking compiled code ... WARNING
File ‘affxparser/libs/affxparser.so’:
  Found ‘_ZSt4cerr’, possibly from ‘std::cerr’ (C++)
    Object: ‘fusion_sdk/util/Verbose.o’
  Found ‘_ZSt4cout’, possibly from ‘std::cout’ (C++)
    Objects: ‘fusion_sdk/file/TsvFile/TsvFile.o’,
      ‘fusion_sdk/util/Err.o’
  Found ‘abort’, possibly from ‘abort’ (C)
    Objects:
      ‘fusion_sdk/calvin_files/fusion/src/CalvinAdapter/CalvinCHPDataAdapter.o’,
... 14 lines ...
    Objects: ‘fusion_sdk/calvin_files/utils/src/AffymetrixGuid.o’,
      ‘fusion_sdk/util/Verbose.o’
  Found ‘srand’, possibly from ‘srand’ (C)
    Object: ‘fusion_sdk/calvin_files/utils/src/AffymetrixGuid.o’
  Found ‘stdout’, possibly from ‘stdout’ (C)
    Object: ‘fusion_sdk/util/Util.o’

Compiled code should not call entry points which might terminate R nor
write to stdout/stderr instead of to the console, nor the system RNG.

See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.

checking for portable file names ... NOTE
Found the following non-portable file paths:
  affxparser/src/fusion_sdk/calvin_files/writers/src/CalvinCHPQuantificationDetectionFileBufferWriter.cpp
  affxparser/src/fusion_sdk/calvin_files/writers/src/CalvinCHPQuantificationDetectionFileBufferWriter.h

Tarballs are only required to store paths of up to 100 bytes and cannot
store those of more than 256 bytes, with restrictions including to 100
bytes for the final component.
See section ‘Package structure’ in the ‘Writing R Extensions’ manual.

checking installed package size ... NOTE
  installed size is 14.5Mb
  sub-directories of 1Mb or more:
    libs  13.9Mb
```

## alphahull (2.1)
Maintainer: Beatriz Pateiro-Lopez <beatriz.pateiro@usc.es>

0 errors | 0 warnings | 0 notes

## arc (1.0)
Maintainer: Tomas Kliegr <kliegr@gmail.com>

0 errors | 0 warnings | 0 notes

## aroma.affymetrix (3.0.0)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/aroma.affymetrix/issues

0 errors | 0 warnings | 0 notes

## aroma.apd (0.6.0)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/aroma.apd/issues

0 errors | 0 warnings | 0 notes

## aroma.cn (1.6.1)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/aroma.cn/issues

0 errors | 0 warnings | 0 notes

## aroma.core (3.0.0)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/aroma.core/issues

0 errors | 0 warnings | 1 note 

```
checking package dependencies ... NOTE
Packages suggested but not available for checking:
  ‘expectile’ ‘HaarSeg’ ‘mpcbs’
```

## aroma.light (3.4.0)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/aroma.light/issues

0 errors | 0 warnings | 1 note 

```
checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  inst/rsp/.rspPlugins
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.
```

## babel (0.3-0)
Maintainer: Adam B. Olshen <adam.olshen@ucsf.edu>

0 errors | 0 warnings | 0 notes

## bedr (1.0.3)
Maintainer: Paul Boutros <Paul.Boutros@oicr.on.ca>

0 errors | 0 warnings | 0 notes

## bsseq (1.10.0)
Maintainer: Kasper Daniel Hansen <kasperdanielhansen@gmail.com>  
Bug reports: https://github.com/kasperdanielhansen/bsseq/issues

0 errors | 0 warnings | 0 notes

## calmate (0.12.1)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/calmate/issues

0 errors | 0 warnings | 0 notes

## Causata (4.2-0)
Maintainer: Justin Hemann <justinh@causata.com>

0 errors | 0 warnings | 5 notes

```
checking DESCRIPTION meta-information ... NOTE
Malformed Title field: should not end in a period.

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘integration_tests’

checking R code for possible problems ... NOTE
BinaryCut: no visible global function definition for ‘quantile’
CleanNaFromContinuousFunction: no visible global function definition
  for ‘median’
CleanNaFromContinuousFunction: no visible global function definition
  for ‘na.omit’
GetMetadata.Connect: no visible global function definition for
  ‘dbGetQuery’
GetRawData.Connect: no visible global function definition for
  ‘dbGetQuery’
... 12 lines ...
  ‘contrasts’
predict.GlmnetModelDefinition: no visible global function definition
  for ‘predict’
Undefined global functions or variables:
  coef contrasts dbGetQuery median model.matrix na.omit predict
  quantile read.csv sd terms.formula
Consider adding
  importFrom("stats", "coef", "contrasts", "median", "model.matrix",
             "na.omit", "predict", "quantile", "sd", "terms.formula")
  importFrom("utils", "read.csv")
to your NAMESPACE file.

checking line endings in Makefiles ... NOTE
Found the following Makefile(s) without a final LF:
  inst/doc/Makefile
Some ‘make’ programs ignore lines not ending in LF.

checking files in ‘vignettes’ ... NOTE
The following files look like leftovers/mistakes:
  ‘Causata-vignette.log’
Please remove them from your package.
The following directory looks like a leftover from 'knitr':
  ‘figure’
Please remove from your package.
```

## ChemoSpec (4.3.34)
Maintainer: Bryan A. Hanson <hanson@depauw.edu>  
Bug reports: https://github.com/bryanhanson/ChemoSpec/issues

0 errors | 0 warnings | 1 note 

```
checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘chemometrics’
```

## CNEr (1.10.1)
Maintainer: Ge Tan <ge.tan09@imperial.ac.uk>  
Bug reports: https://github.com/ge11232002/CNEr/issues

0 errors | 2 warnings | 2 notes

```
checking compiled code ... WARNING
File ‘CNEr/libs/CNEr.so’:
  Found ‘abort’, possibly from ‘abort’ (C)
    Object: ‘ucsc/errabort.o’
  Found ‘exit’, possibly from ‘exit’ (C)
    Objects: ‘ucsc/errabort.o’, ‘ucsc/pipeline.o’
  Found ‘puts’, possibly from ‘printf’ (C), ‘puts’ (C)
    Object: ‘ucsc/pipeline.o’
  Found ‘rand’, possibly from ‘rand’ (C)
    Object: ‘ucsc/obscure.o’
  Found ‘stderr’, possibly from ‘stderr’ (C)
    Objects: ‘ucsc/axt.o’, ‘ucsc/errabort.o’, ‘ucsc/obscure.o’,
      ‘ucsc/verbose.o’, ‘ucsc/os.o’
  Found ‘stdout’, possibly from ‘stdout’ (C)
    Objects: ‘ucsc/common.o’, ‘ucsc/errabort.o’, ‘ucsc/verbose.o’,
      ‘ucsc/os.o’

Compiled code should not call entry points which might terminate R nor
write to stdout/stderr instead of to the console, nor the system RNG.

See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning: It seems you should call rmarkdown::render() instead of knitr::knit2html() because CNEr.Rmd appears to be an R Markdown v2 document.
Quitting from lines 2-15 (CNEr.Rmd) 
Error: processing vignette 'CNEr.Rmd' failed with diagnostics:
could not find function "doc_date"
Execution halted


checking installed package size ... NOTE
  installed size is 18.2Mb
  sub-directories of 1Mb or more:
    extdata  15.9Mb
    libs      1.1Mb

checking dependencies in R code ... NOTE
Unexported objects imported by ':::' calls:
  ‘BiocGenerics:::replaceSlots’ ‘S4Vectors:::make_zero_col_DataFrame’
  See the note in ?`:::` about the use of this operator.
```

## CollapsABEL (0.10.8)
Maintainer: Kaiyin Zhong <kindlychung@gmail.com>  
Bug reports: https://bitbucket.org/kindlychung/collapsabel2/issues

0 errors | 0 warnings | 0 notes

## DeepBlueR (1.0.3)
Maintainer: Felipe Albrecht <felipe.albrecht@mpi-inf.mpg.de>, Markus List <markus.list@mpi-inf.mpg.de>

0 errors | 0 warnings | 1 note 

```
checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘logo.png’
```

## DEGraph (1.26.0)
Maintainer: Laurent Jacob <laurent.jacob@gmail.com>

0 errors | 0 warnings | 6 notes

```
checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .BBSoptions
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.

checking DESCRIPTION meta-information ... NOTE
Packages listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘graph’ ‘KEGGgraph’ ‘lattice’ ‘RBGL’ ‘rrcov’ ‘Rgraphviz’ ‘NCIgraph’
A package should be listed in only one of these fields.

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘incl’

checking dependencies in R code ... NOTE
Package in Depends field not imported from: ‘R.utils’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.
Unexported object imported by a ':::' call: ‘graph:::getIndices’
  See the note in ?`:::` about the use of this operator.

checking R code for possible problems ... NOTE
AN.test: no visible global function definition for ‘var’
BS.test: no visible global function definition for ‘pnorm’
getConnectedComponentList: no visible binding for global variable
  ‘Arguments’
getConnectedComponentList: no visible global function definition for
  ‘pushState’
getConnectedComponentList: no visible global function definition for
  ‘popState’
getKEGGPathways: no visible binding for global variable ‘Arguments’
... 65 lines ...
  Arguments ProgressBar as enter exit heat.colors increase na.omit
  phyper pnorm popState pushState quantile reset rnorm str
  translateNCI2GeneID var
Consider adding
  importFrom("grDevices", "heat.colors")
  importFrom("methods", "as")
  importFrom("stats", "na.omit", "phyper", "pnorm", "quantile", "rnorm",
             "var")
  importFrom("utils", "str")
to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
contains 'methods').

checking Rd line widths ... NOTE
Rd file 'getKEGGPathways.Rd':
  \usage lines wider than 90 characters:
     getKEGGPathways(path=NULL, rootPath="networkData/ftp.genome.jp/pub/kegg/xml/kgml", organism="hsa", metaTag=c("non-metabolic", "metaboli ... [TRUNCATED]

Rd file 'getSignedGraph.Rd':
  \usage lines wider than 90 characters:
     getSignedGraph(graph, positiveInteractionLabels=c("activation", "expression"), negativeInteractionLabels=c("inhibition", "repression"), ... [TRUNCATED]

Rd file 'laplacianFromA.Rd':
... 10 lines ...

Rd file 'testOneGraph.Rd':
  \examples lines wider than 100 characters:
     pvg <- plotValuedGraph(gr, values=shift, nodeLabels=dn, qMax=0.95, colorPalette=pal, height=40, lwd=1, verbose=TRUE, cex=0.5)
       image.plot(legend.only=TRUE, zlim=range(pvg$breaks), col=pal, legend.shrink=0.3, legend.width=0.8, legend.lab="t-scores", legend.mar= ... [TRUNCATED]

Rd file 'writeAdjacencyMatrix2KGML.Rd':
  \usage lines wider than 90 characters:
     writeAdjacencyMatrix2KGML(mat, pathname, nodePrefix="n", overwrite=FALSE, ..., verbose=FALSE)

These lines will be truncated in the PDF manual.
```

## demi (1.1.2)
Maintainer: Sten Ilmjarv <sten.ilmjarv@gmail.com>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
DEMIPathway: no visible global function definition for ‘dhyper’
DEMIPathway: no visible global function definition for ‘p.adjust’
calcHypergeoExon: no visible global function definition for ‘dhyper’
calcHypergeoProb: no visible global function definition for ‘dhyper’
demi: no visible global function definition for ‘write.table’
demi.t.test: no visible global function definition for ‘t.test’
demi.wilcox.test: no visible global function definition for
  ‘wilcox.test’
demi.wilcox.test.fast: no visible global function definition for
... 10 lines ...
loadCytoband,DEMIExperiment-environment: no visible global function
  definition for ‘data’
loadPathway,DEMIExperiment-environment: no visible global function
  definition for ‘data’
Undefined global functions or variables:
  combn data dhyper median p.adjust t.test wilcox.test write.table
Consider adding
  importFrom("stats", "dhyper", "median", "p.adjust", "t.test",
             "wilcox.test")
  importFrom("utils", "combn", "data", "write.table")
to your NAMESPACE file.
```

## downsize (0.2.1)
Maintainer: William Landau <will.landau@gmail.com>  
Bug reports: https://github.com/wlandau/downsize/issues

0 errors | 0 warnings | 0 notes

## dplR (1.6.4)
Maintainer: Andy Bunn <andy.bunn@wwu.edu>

0 errors | 0 warnings | 0 notes

## DupChecker (1.12.0)
Maintainer: "Quanhu SHENG" <shengqh@gmail.com>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
arrayExpressDownload: no visible global function definition for
  ‘download.file’
geoDownload: no visible global function definition for ‘download.file’
geoDownload: no visible global function definition for ‘untar’
getFtpFilenames: no visible global function definition for ‘read.table’
Undefined global functions or variables:
  download.file read.table untar
Consider adding
  importFrom("utils", "download.file", "read.table", "untar")
to your NAMESPACE file.
```

## edgar (1.0.6)
Maintainer: Gunratan Lonare <lonare.gunratan@gmail.com>

0 errors | 0 warnings | 0 notes

## eegc (1.0.0)
Maintainer: Xiaoyuan Zhou <zhouxiaoyuan@picb.ac.cn>

0 errors | 0 warnings | 4 notes

```
checking installed package size ... NOTE
  installed size is 11.1Mb
  sub-directories of 1Mb or more:
    data  10.5Mb

checking R code for possible problems ... NOTE
categorizeGene : <anonymous>: no visible global function definition for
  ‘quantile’
densityPlot: no visible binding for global variable ‘density’
densityPlot: no visible global function definition for ‘lines’
densityPlot: no visible global function definition for ‘axis’
densityPlot: no visible global function definition for ‘adjustcolor’
densityPlot: no visible global function definition for ‘legend’
densityPlot : <anonymous>: no visible global function definition for
  ‘quantile’
... 26 lines ...
  abline adjustcolor axis colorRampPalette control density dev.copy2pdf
  legend lines lm model.matrix p.adjust par phyper points quantile
  results text title treat
Consider adding
  importFrom("grDevices", "adjustcolor", "colorRampPalette",
             "dev.copy2pdf")
  importFrom("graphics", "abline", "axis", "legend", "lines", "par",
             "points", "text", "title")
  importFrom("stats", "density", "lm", "model.matrix", "p.adjust",
             "phyper", "quantile")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'markerScatter.Rd':
  \usage lines wider than 90 characters:
     col = NULL, xlab = NULL, ylab = NULL, main = NULL, add.line = TRUE, text.cex = 1, legend.labels = NULL,

These lines will be truncated in the PDF manual.

checking for unstated dependencies in vignettes ... NOTE
'::' or ':::' import not declared from: ‘BiocStyle’
```

## EEM (1.1.1)
Maintainer: Vipavee Trivittayasil <vipavee.tri@gmail.com>

0 errors | 0 warnings | 0 notes

## eiCompare (1.7)
Maintainer: Loren Collingwood <loren.collingwood@ucr.edu>

0 errors | 0 warnings | 0 notes

## ELMER (1.4.0)
Maintainer: Simon Coetzee <Simon.Coetzee@cshs.org>

0 errors | 0 warnings | 3 notes

```
checking dependencies in R code ... NOTE
Packages in Depends field not imported from:
  ‘ELMER.data’ ‘Homo.sapiens’
  ‘IlluminaHumanMethylation450kanno.ilmn12.hg19’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking R code for possible problems ... NOTE
Stat.diff.meth: no visible binding for global variable ‘t.test’
Stat.nonpara : <anonymous>: no visible global function definition for
  ‘wilcox.test’
Stat.nonpara: no visible global function definition for ‘wilcox.test’
Stat.nonpara.permu : <anonymous>: no visible global function definition
  for ‘wilcox.test’
Stat.nonpara.permu: no visible global function definition for
  ‘write.table’
TCGA.pipe: no visible global function definition for ‘read.csv’
... 52 lines ...
Undefined global functions or variables:
  Homo.sapiens IlluminaHumanMethylation450kanno.ilmn12.hg19 OR coef
  data dev.off label lm lowerOR motif p.adjust pdf pvalue read.csv
  read.delim read.table str t.test upperOR value wilcox.test write.csv
  write.table
Consider adding
  importFrom("grDevices", "dev.off", "pdf")
  importFrom("stats", "coef", "lm", "p.adjust", "t.test", "wilcox.test")
  importFrom("utils", "data", "read.csv", "read.delim", "read.table",
             "str", "write.csv", "write.table")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'fetch.mee.Rd':
  \usage lines wider than 90 characters:
     fetch.mee(meth, exp, sample, probeInfo, geneInfo, probes = NULL, genes = NULL, TCGA = FALSE)

Rd file 'get.diff.meth.Rd':
  \usage lines wider than 90 characters:
     get.diff.meth(mee, diff.dir = "hypo", cores = NULL, percentage = 0.2, pvalue = 0.01, sig.dif = 0.3, dir.out = "./", save = TRUE)

Rd file 'get.enriched.motif.Rd':
  \usage lines wider than 90 characters:
     get.enriched.motif(probes.motif, probes, background.probes, lower.OR = 1.1, min.incidence = 10, 

Rd file 'get.feature.probe.Rd':
  \usage lines wider than 90 characters:
     get.feature.probe(feature, TSS, TSS.range = list(upstream = 2000, downstream = 2000), promoter = FALSE, rm.chr = NULL)

These lines will be truncated in the PDF manual.
```

## eudysbiome (1.4.0)
Maintainer: Xiaoyuan Zhou <zhouxiaoyuan@picb.ac.cn>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
Cartesian: no visible global function definition for ‘adjustcolor’
Cartesian: no visible global function definition for ‘colors’
Cartesian: no visible global function definition for ‘dev.hold’
Cartesian: no visible global function definition for ‘dev.flush’
Cartesian: no visible global function definition for ‘plot’
Cartesian: no visible global function definition for ‘points’
Cartesian: no visible global function definition for ‘abline’
Cartesian: no visible global function definition for ‘axis’
Cartesian: no visible global function definition for ‘text’
... 21 lines ...
  importFrom("graphics", "abline", "axis", "mtext", "par", "plot",
             "points", "rect", "text", "title")
  importFrom("stats", "chisq.test")
  importFrom("utils", "combn", "data", "download.file", "read.table",
             "unzip")
to your NAMESPACE file.

Found the following calls to data() loading into the global environment:
File ‘eudysbiome/R/microAnnotate.R’:
  data(harmGenera)
See section ‘Good practice’ in ‘?data’.
```

## evclust (1.0.3)
Maintainer: Thierry Denoeux <tdenoeux@utc.fr>

0 errors | 0 warnings | 0 notes

## ezknitr (0.6)
Maintainer: Dean Attali <daattali@gmail.com>  
Bug reports: https://github.com/ropenscilabs/ezknitr/issues

0 errors | 0 warnings | 0 notes

## FGNet (3.8.0)
Maintainer: Sara Aibar <saibar@usal.es>

1 error  | 1 warning  | 3 notes

```
checking examples ... ERROR
Running examples in ‘FGNet-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: fea2incidMat
> ### Title: Transforms FEA output into incidence matrices.
> ### Aliases: fea2incidMat toMatrix adjMatrix
> 
> ### ** Examples
... 49 lines ...
> 
> functionalNetwork(incidMat)
>     
> # Term-based network
> incidMatTerms <- fea2incidMat(results, key="Terms")
> functionalNetwork(incidMatTerms, plotOutput="dynamic")
Warning: no DISPLAY variable so Tk is not available
Error in structure(.External(.C_dotTclObjv, objv), class = "tclObj") : 
  [tcl] invalid command name "font".
Calls: functionalNetwork ... .tkplot.convert.font -> <Anonymous> -> tcl -> .Tcl.objv -> structure
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

    simplify, union

The following objects are masked from 'package:S4Vectors':

    compare, union

... 8 lines ...

The following object is masked from 'package:base':

    union

The following metagroups/clusters have been filtered out: 1, 3, 4
Warning: no DISPLAY variable so Tk is not available
Quitting from lines 500-504 (FGNet.Rmd) 
Error: processing vignette 'FGNet.Rmd' failed with diagnostics:
[tcl] invalid command name "font".
Execution halted

checking installed package size ... NOTE
  installed size is  7.7Mb
  sub-directories of 1Mb or more:
    doc   7.1Mb

checking R code for possible problems ... NOTE
FGNet_GUI: no visible global function definition for ‘data’
FGNet_GUI: no visible global function definition for ‘gtkWindowNew’
FGNet_GUI: no visible global function definition for ‘gtkStatusbar’
FGNet_GUI: no visible global function definition for ‘gtkNotebookNew’
FGNet_GUI: no visible global function definition for ‘setNames’
FGNet_GUI: no visible global function definition for
  ‘gtkNotebookInsertPage’
FGNet_GUI: no visible global function definition for ‘gtkLabelNew’
FGNet_GUI: no visible global function definition for ‘gtkVBoxNew’
... 552 lines ...
Consider adding
  importFrom("grDevices", "col2rgb", "dev.off", "png", "rainbow")
  importFrom("graphics", "abline", "axis", "barplot", "box", "boxplot",
             "image", "layout", "legend", "par", "plot", "title")
  importFrom("methods", "new")
  importFrom("stats", "dist", "hclust", "quantile", "setNames")
  importFrom("utils", "View", "browseURL", "data", "download.file",
             "installed.packages", "read.csv", "read.table",
             "write.table")
to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
contains 'methods').

checking Rd line widths ... NOTE
Rd file 'format_david.Rd':
  \usage lines wider than 90 characters:
     format_david(fileName, jobName = NULL, geneLabels = NULL, moveFile = FALSE, downloadFile=TRUE)

Rd file 'functionalNetwork.Rd':
  \examples lines wider than 100 characters:
     functionalNetwork(incidMat, plotType="bipartite", plotOutput="dynamic", vSize=c(default=10, GLC7=20, PTA1=20))

These lines will be truncated in the PDF manual.
```

## freesurfer (1.0)
Maintainer: John Muschelli <muschellij2@gmail.com>  
Bug reports: https://github.com/muschellij2/freesurfer/issues

0 errors | 0 warnings | 0 notes

## fslr (2.0)
Maintainer: John Muschelli <muschellij2@gmail.com>  
Bug reports: https://github.com/muschellij2/fslr/issues

0 errors | 0 warnings | 0 notes

## future.BatchJobs (0.13.1)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/future.BatchJobs/issues

0 errors | 0 warnings | 0 notes

## gdalUtils (2.0.1.7)
Maintainer: Jonathan Asher Greenberg <gdalUtils@estarcion.net>

0 errors | 0 warnings | 0 notes

## gofCopula (0.2-3)
Maintainer: Simon Trimborn <simon.trimborn@wiwi.hu-berlin.de>

0 errors | 0 warnings | 0 notes

## hashFunction (1.0)
Maintainer: Xiaowei Zhan <zhanxw@gmail.com>

0 errors | 0 warnings | 2 notes

```
checking package namespace information ... NOTE
  Namespace with empty importFrom: ‘utils’

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘test’
```

## hoardeR (0.9.2)
Maintainer: Daniel Fischer <daniel.fischer@luke.fi>

0 errors | 0 warnings | 0 notes

## HyPhy (1.0)
Maintainer: Nathaniel Malachi Hallinan <nmhallinan@gmail.com>

0 errors | 0 warnings | 3 notes

```
checking dependencies in R code ... NOTE
Packages in Depends field not imported from:
  ‘R.utils’ ‘ape’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking R code for possible problems ... NOTE
requiprobable: no visible global function definition for ‘extract’
requiprobable : make.tree: no visible global function definition for
  ‘extract’
requiprobable : make.tree: no visible global function definition for
  ‘runif’
rgenetree: no visible global function definition for ‘dbinom’
rgenetree: no visible global function definition for ‘dnbinom’
rgenetree: no visible global function definition for ‘runif’
rgenetree: no visible global function definition for ‘rbinom’
rgenetree: no visible global function definition for ‘rnbinom’
Undefined global functions or variables:
  dbinom dnbinom extract rbinom rnbinom runif
Consider adding
  importFrom("stats", "dbinom", "dnbinom", "rbinom", "rnbinom", "runif")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'requiprobable.Rd':
  \examples lines wider than 100 characters:
     for (i in 1:10) unlabeled[[i]]$tip.label<-sample(c("A1","A2","B1","B2","B3","C1","C2","C3","C4","C5"))

Rd file 'rgenetree.Rd':
  \examples lines wider than 100 characters:
     ##Now lets start with 3 genes, set the number of genes at each tip of spec and vary mu between the branches of spec 

These lines will be truncated in the PDF manual.
```

## joinXL (1.0.1)
Maintainer: Yvonne Glanville <yvonneglanville@gmail.com>  
Bug reports: http://github.com/yvonneglanville/joinXL/issues

0 errors | 0 warnings | 0 notes

## listenv (0.6.0)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/listenv/issues

0 errors | 0 warnings | 0 notes

## LSPFP (1.0.0)
Maintainer: Rafael Dellen <Rafael.Dellen@uni-duesseldorf.de>

0 errors | 0 warnings | 0 notes

## mailR (0.4.1)
Maintainer: Rahul Premraj <r.premraj+mailR@gmail.com>

0 errors | 0 warnings | 0 notes

## mcparallelDo (1.1.0)
Maintainer: Russell S. Pierce <russell.s.pierce@gmail.com>  
Bug reports: https://github.com/drknexus/mcparallelDo/issues

0 errors | 0 warnings | 0 notes

## methylKit (1.0.0)
Maintainer: Altuna Akalin <aakalin@gmail.com>

0 errors | 0 warnings | 0 notes

## MPAgenomics (1.1.2)
Maintainer: Samuel Blanck <samuel.blanck@inria.fr>

0 errors | 0 warnings | 2 notes

```
checking dependencies in R code ... NOTE
'library' or 'require' calls in package code:
  ‘R.devices’ ‘R.filesets’ ‘R.methodsS3’ ‘R.oo’ ‘aroma.affymetrix’
  ‘aroma.cn’ ‘aroma.core’ ‘aroma.light’ ‘matrixStats’ ‘snowfall’
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
Unexported object imported by a ':::' call: ‘cghseg:::segmeanCO’
  See the note in ?`:::` about the use of this operator.

checking R code for possible problems ... NOTE
.varregtimescount: no visible global function definition for ‘var’
CGHSEGaroma: no visible global function definition for ‘read.csv’
CGHSEGaroma : <anonymous>: no visible global function definition for
  ‘points’
CGHSEGaroma : <anonymous>: no visible global function definition for
  ‘lines’
CGHSEGaroma : <anonymous>: no visible global function definition for
  ‘write.table’
CGHcall: no visible global function definition for ‘mad’
... 43 lines ...
tumorboostPlot: no visible global function definition for ‘par’
tumorboostPlot: no visible global function definition for ‘axis’
tumorboostPlot: no visible global function definition for ‘points’
Undefined global functions or variables:
  axis head lines lm mad median optim par points read.csv sd var
  write.table
Consider adding
  importFrom("graphics", "axis", "lines", "par", "points")
  importFrom("stats", "lm", "mad", "median", "optim", "sd", "var")
  importFrom("utils", "head", "read.csv", "write.table")
to your NAMESPACE file.
```

## MSIseq (1.0.0)
Maintainer: Mini Huang <mini.huang@nus.edu.sg>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
MSIseq.classify: no visible global function definition for ‘predict’
Undefined global functions or variables:
  predict
Consider adding
  importFrom("stats", "predict")
to your NAMESPACE file.
```

## neurobase (1.1.1.1)
Maintainer: John Muschelli <muschellij2@gmail.com>  
Bug reports: https://github.com/muschellij2/nifti/issues

0 errors | 0 warnings | 0 notes

## NSA (0.0.32)
Maintainer: Maria Ortiz-Estevez <mortizest@gmail.com>

0 errors | 0 warnings | 6 notes

```
checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘R.methodsS3’ ‘MASS’ ‘matrixStats’ ‘R.oo’ ‘R.utils’ ‘aroma.core’
  ‘aroma.affymetrix’ ‘DNAcopy’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘incl’

checking dependencies in R code ... NOTE
Packages in Depends field not imported from:
  ‘DNAcopy’ ‘MASS’ ‘R.methodsS3’ ‘R.oo’ ‘aroma.affymetrix’ ‘aroma.core’
  ‘matrixStats’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  NSAByTotalAndFracB.matrix allocateOutputDataSets.NSANormalization
  allocateOutputDataSets.SNPsNormalization
  allocateOutputDataSets.SampleNormalization
  as.character.NSANormalization as.character.SNPsNormalization
  as.character.SampleNormalization findArraysTodo.NSANormalization
  findArraysTodo.SampleNormalization findUnitsTodo.SNPsNormalization
  fitNSA.matrix fitNSAcnPs.matrix getDataSets.NSANormalization
  getDataSets.SNPsNormalization getDataSets.SampleNormalization
  getName.NSANormalization getName.SNPsNormalization
  getName.SampleNormalization getOutputDataSets.NSANormalization
  getOutputDataSets.SNPsNormalization
  getOutputDataSets.SampleNormalization getPath.NSANormalization
  getPath.SNPsNormalization getPath.SampleNormalization
  getRootPath.NSANormalization getRootPath.SNPsNormalization
  getRootPath.SampleNormalization process.NSANormalization
  process.SNPsNormalization process.SampleNormalization
  sampleNByTotalAndFracB.numeric snpsNByTotalAndFracB.matrix
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking R code for possible problems ... NOTE
NB: .First.lib is obsolete and will not be used in R >= 3.0.0

.First.lib: no visible global function definition for
  ‘packageDescription’
NSAByTotalAndFracB.matrix: no visible global function definition for
  ‘throw’
NSAByTotalAndFracB.matrix: no visible global function definition for
  ‘str’
NSANormalization: no visible global function definition for ‘throw’
... 279 lines ...
  extractMatrix findUnitsTodo getAsteriskTags getChipType getFile
  getFullName getFullNames getGenomeInformation getName getNames
  getPath getPathname getPathnames getPositions getRam getRootPath
  getTags getUnitsOnChromosome hist median nbrOfFiles newInstance
  packageDescription rowAlls rowMedians segment setTags str throw trim
  verbose
Consider adding
  importFrom("graphics", "hist")
  importFrom("stats", "approxfun", "median")
  importFrom("utils", "packageDescription", "str")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'NSANormalization.Rd':
  \examples lines wider than 100 characters:
     by <- 50e3; # 50kb bins; you may want to try with other amounts of smoothing xOut <- seq(from=xRange[1], to=xRange[2], by=by);
     plot(getSignals(cnCNPS), getSignals(cnSNPS), xlim=Clim, ylim=Clim); abline(a=0, b=1, col="red", lwd=2);

These lines will be truncated in the PDF manual.
```

## oce (0.9-19)
Maintainer: Dan Kelley <Dan.Kelley@Dal.Ca>  
Bug reports: https://github.com/dankelley/oce/issues

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  5.3Mb
  sub-directories of 1Mb or more:
    help   2.0Mb
```

## oneChannelGUI (1.40.0)
Maintainer: Raffaele A Calogero <raffaele.calogero@unito.it>

0 errors | 2 warnings | 6 notes

```
checking whether package ‘oneChannelGUI’ can be installed ... WARNING
Found the following significant warnings:
  Warning: no DISPLAY variable so Tk is not available
  Warning: loading Rplot failed
See ‘/home/hb/repositories/R.utils/revdep/checks/oneChannelGUI.Rcheck/00install.out’ for details.

checking sizes of PDF files under ‘inst/doc’ ... WARNING
  ‘gs+qpdf’ made some significant size reductions:
     compacted ‘Exon-level.analysis.pdf’ from 1395Kb to 598Kb
     compacted ‘RNAseq.pdf’ from 1979Kb to 385Kb
  consider running tools::compactPDF(gs_quality = "ebook") on these files

checking package dependencies ... NOTE
Depends: includes the non-default packages:
  ‘Biobase’ ‘affylmGUI’ ‘tkrplot’ ‘tkWidgets’ ‘IRanges’ ‘Rsamtools’
  ‘Biostrings’ ‘siggenes’ ‘chimera’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

checking installed package size ... NOTE
  installed size is  6.6Mb
  sub-directories of 1Mb or more:
    doc   5.0Mb

checking DESCRIPTION meta-information ... NOTE
Malformed Description field: should contain one or more complete sentences.

checking dependencies in R code ... NOTE
'library' or 'require' calls in package code:
  ‘BSgenome.Hsapiens.UCSC.hg19’ ‘BSgenome.Mmusculus.UCSC.mm9’
  ‘BSgenome.Rnorvegicus.UCSC.rn4’ ‘Genominator’ ‘affy’ ‘affyPLM’
  ‘chipseq’ ‘maSigPro’
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.

checking R code for possible problems ... NOTE
.consistentFilter: warning in get("midas.p.Available", env =
  affylmGUIenvironment): partial argument match of 'env' to 'envir'
.consistentFilter: warning in get("AltSplRP.e.Available", env =
  affylmGUIenvironment): partial argument match of 'env' to 'envir'
.consistentFilter: warning in get("AltSplRP.e.p", env =
  affylmGUIenvironment): partial argument match of 'env' to 'envir'
.consistentFilter: warning in get("midas.p", env =
  affylmGUIenvironment): partial argument match of 'env' to 'envir'
.consistentFilter: warning in get("spliceIndexData", env =
... 2499 lines ...
  data("chrLength", package = "oneChannelGUI")
  data("chrLength", package = "oneChannelGUI")
  data("chrLength", package = "oneChannelGUI")
  data("chrLength", package = "oneChannelGUI")
File ‘oneChannelGUI/R/generaltoolsmenu.R’:
  data(HuExExonProbesetLocation)
  data(MoExExonProbesetLocation)
  data(RaExExonProbesetLocation)
File ‘oneChannelGUI/R/standalonefunctions.R’:
  data("chrLength", package = "oneChannelGUI")
See section ‘Good practice’ in ‘?data’.

checking Rd line widths ... NOTE
Rd file 'oneChannelGUI.Rd':
  \usage lines wider than 90 characters:
     wrapperMirnaCounts(working.dir, out.dir, org = "hsa", threads = 1, cutadapt.path = "/usr/local/bin/cutadapt", parallel = FALSE, ...)

Rd file 'standAloneBuildingLocalAnnotation.Rd':
  \usage lines wider than 90 characters:
        standAloneBuildingLocalAnnotation(libDirLocation = getwd(), netaffxUser = "myemail@somewhere.org", netaffxUserPw = "yourpassword", w ... [TRUNCATED]

These lines will be truncated in the PDF manual.
```

## openCyto (1.12.1)
Maintainer: Mike Jiang <wjiang2@fhcrc.org>

0 errors | 2 warnings | 5 notes

```
checking dependencies in R code ... WARNING
'library' or 'require' call to ‘parallel’ in package code.
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
Unexported objects imported by ':::' calls:
  ‘flowClust:::.ellipsePoints’ ‘flowStats:::drvkde’
  ‘flowStats:::warpSetNCDF’ ‘flowWorkspace:::.cpp_addGate’
  ‘flowWorkspace:::.getAllDescendants’ ‘flowWorkspace:::trimWhiteSpace’
  ‘lattice:::updateList’
  See the note in ?`:::` about the use of this operator.
  Including base/recommended package(s):
  ‘lattice’
There are ::: calls to the package's namespace in its code. A package
  almost never needs to use ::: for its own objects:
  ‘groupBy’ ‘isCollapse’ ‘ppMethod’ ‘unlockNamespace’

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning: Removed 14 rows containing missing values (geom_hex).
Warning: Removed 7 rows containing missing values (geom_hex).
Warning: Removed 7 rows containing missing values (geom_hex).
Warning: Removed 42 rows containing missing values (geom_hex).
Warning in rlm.default(x, y, weights, method = method, wt.method = wt.method,  :
  'rlm' failed to converge in 5 steps
Warning in singletGate(fr, area = channels[1], height = channels[2], ...) :
... 8 lines ...
Warning: Removed 163 rows containing non-finite values (stat_binhex).
Warning: Removed 13 rows containing missing values (geom_hex).
loading R object...
loading tree object...
Done
The prior specification has no effect when usePrior=no
Using the serial version of flowClust
Quitting from lines 155-161 (HowToAutoGating.Rmd) 
Error: processing vignette 'HowToAutoGating.Rmd' failed with diagnostics:
Names missing in input list.
Execution halted

checking DESCRIPTION meta-information ... NOTE
Package listed in more than one of Depends, Imports, Suggests, Enhances:
  ‘flowWorkspace’
A package should be listed in only one of these fields.

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘example’

checking R code for possible problems ... NOTE
Found the following possibly unsafe calls:
File ‘openCyto/R/pluginFramework.R’:
  unlockBinding(methodName, ENV)
  unlockBinding(methodName, ENV)

.boundary: no visible global function definition for ‘rectangleGate’
.center_mode: no visible global function definition for ‘density’
.find_peaks: no visible global function definition for ‘density’
.find_peaks: no visible global function definition for ‘points’
... 236 lines ...
  importFrom("grDevices", "rainbow")
  importFrom("graphics", "abline", "hist", "legend", "lines", "par",
             "points", "text")
  importFrom("methods", "as", "callNextMethod", "extends", "new",
             "selectMethod")
  importFrom("stats", "cov", "cov.wt", "cutree", "density", "dist",
             "dnorm", "embed", "filter", "hclust", "kmeans", "median",
             "na.omit", "predict", "pt", "qchisq", "qf", "quantile",
             "sd", "smooth.spline", "uniroot", "var", "weighted.mean")
to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
contains 'methods').

checking Rd line widths ... NOTE
Rd file 'add_pop.Rd':
  \examples lines wider than 100 characters:
      add_pop(gs, gating_method = "mindensity", dims = "CCR7,CD45RA", parent = "cd4-cd8+", pop = "CCR7+/-CD45RA+/-")
     add_pop(gs, gating_method = "polyFunctions", parent = "cd8", gating_args = "cd8/IFNg:cd8/IL2:cd8/TNFa")
     add_pop(gs, alias = "IL2orIFNg", gating_method = "boolGate", parent = "cd4", gating_args = "cd4/IL2|cd4/IFNg") 

These lines will be truncated in the PDF manual.

checking for unstated dependencies in vignettes ... NOTE
'library' or 'require' call not declared from: ‘ggcyto’
```

## paxtoolsr (1.8.0)
Maintainer: Augustin Luna <lunaa@cbio.mskcc.org>

0 errors | 0 warnings | 2 notes

```
checking installed package size ... NOTE
  installed size is 30.9Mb
  sub-directories of 1Mb or more:
    extdata   5.6Mb
    java     24.5Mb

checking R code for possible problems ... NOTE
.onLoad: no visible global function definition for ‘write.table’
downloadFile: no visible global function definition for ‘URLencode’
downloadFile: no visible global function definition for ‘read.table’
downloadFile: no visible global function definition for ‘write.table’
downloadPc2: no visible global function definition for ‘select.list’
downloadSignedPC: no visible global function definition for
  ‘read.table’
getPcRequest: no visible global function definition for ‘URLencode’
readSifnx: no visible global function definition for ‘read.table’
splitSifnxByPathway: no visible global function definition for
  ‘txtProgressBar’
splitSifnxByPathway: no visible global function definition for
  ‘%dopar%’
splitSifnxByPathway: no visible global function definition for
  ‘setTxtProgressBar’
Undefined global functions or variables:
  %dopar% URLencode read.table select.list setTxtProgressBar
  txtProgressBar write.table
Consider adding
  importFrom("utils", "URLencode", "read.table", "select.list",
             "setTxtProgressBar", "txtProgressBar", "write.table")
to your NAMESPACE file.
```

## PGA (1.4.0)
Maintainer: Bo Wen <wenbo@genomics.cn>, Shaohang Xu <xsh.skye@gmail.com>

0 errors | 0 warnings | 3 notes

```
checking installed package size ... NOTE
  installed size is  5.6Mb
  sub-directories of 1Mb or more:
    extdata   1.8Mb

checking dependencies in R code ... NOTE
Unexported objects imported by ':::' calls:
  ‘biomaRt:::martBM’ ‘biomaRt:::martDataset’ ‘biomaRt:::martHost’
  ‘customProDB:::makeTranscriptDbFromBiomart_archive’
  See the note in ?`:::` about the use of this operator.

checking R code for possible problems ... NOTE
.base_transfer: no visible binding for global variable ‘peptide’
.base_transfer: no visible binding for global variable ‘refbase’
.base_transfer: no visible binding for global variable ‘varbase’
.base_transfer: no visible binding for global variable ‘aaref’
.base_transfer: no visible binding for global variable ‘aavar’
.base_transfer: no visible binding for global variable ‘Type’
.base_transfer: no visible binding for global variable ‘Freq’
.get_30aa_splited_seq: no visible global function definition for ‘.’
.get_30aa_splited_seq: no visible binding for global variable ‘id’
... 216 lines ...
reportSNV: no visible binding for global variable ‘abc’
reportSNV: no visible binding for global variable ‘xyz’
Undefined global functions or variables:
  . .I .N .SD CUFF_ID Change Class Evalue Frame Freq ID Index Mass
  MutNum Query Qvalue Strand Substring Type aapos aaref aavar abc
  alleleCount alleles charge chr chrom cumlen delta_da delta_ppm evalue
  gene_name genename genome<- id isSAP isUnique junType jun_type label
  miss mods mrnaAcc mz name output pep peptide pincoding position
  pro_name proname prot protAcc protein rbindlist readAAStringSet
  readDNAStringSet refbase rsid seqlengths seqlevels seqlevels<- subseq
  transcript tx_name txid txname varbase writeXStringSet x xyz y
```

## PopGenReport (2.2.2)
Maintainer: Bernd Gruber <bernd.gruber@canberra.edu.au>

0 errors | 0 warnings | 1 note 

```
checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘ecodist’
```

## proteoQC (1.10.0)
Maintainer: Bo Wen <wenbo@genomics.cn>

0 errors | 0 warnings | 4 notes

```
checking installed package size ... NOTE
  installed size is  5.5Mb
  sub-directories of 1Mb or more:
    extdata   4.0Mb

checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  print.msQCres
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking R code for possible problems ... NOTE
calcMSQCMetrics : <anonymous>: warning in readMSData(x, msLevel = 1):
  partial argument match of 'msLevel' to 'msLevel.'
calcMSQCMetrics : <anonymous>: warning in readMSData(x, msLevel = 2):
  partial argument match of 'msLevel' to 'msLevel.'
addSummaryChart: no visible global function definition for ‘par’
addSummaryChart: no visible global function definition for ‘read.delim’
addSummaryChart: no visible global function definition for ‘png’
addSummaryChart: no visible binding for global variable ‘peplength’
addSummaryChart: no visible binding for global variable ‘..count..’
... 188 lines ...
  importFrom("graphics", "box", "hist", "lines", "par")
  importFrom("stats", "density", "quantile", "sd")
  importFrom("utils", "read.delim", "read.table", "write.table")
to your NAMESPACE file.

Found the following assignments to the global environment:
File ‘proteoQC/R/report.R’:
  assign(x = ".PepSummaryColClass", value = .PepSummaryColClass, 
    envir = .GlobalEnv)
  assign(x = ".ProSummaryColClass", value = .ProSummaryColClass, 
    envir = .GlobalEnv)

checking compiled code ... NOTE
File ‘proteoQC/libs/proteoQC.so’:
  Found ‘_ZSt4cout’, possibly from ‘std::cout’ (C++)
    Object: ‘utilities.o’

Compiled code should not call entry points which might terminate R nor
write to stdout/stderr instead of to the console, nor the system RNG.

See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
```

## PSCBS (0.61.0)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/PSCBS/issues

0 errors | 0 warnings | 0 notes

## psichomics (1.0.0)
Maintainer: Nuno Saraiva Agostinho <nunodanielagostinho@gmail.com>  
Bug reports: https://github.com/nuno-agostinho/psichomics/issues

0 errors | 1 warning  | 1 note 

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in readLines(con) :
  incomplete final line found on 'AS_events_preparation.Rmd'
Warning: It seems you should call rmarkdown::render() instead of knitr::knit2html() because AS_events_preparation.Rmd appears to be an R Markdown v2 document.
Warning in readLines(con) :
  incomplete final line found on 'CLI_tutorial.Rmd'
Warning: It seems you should call rmarkdown::render() instead of knitr::knit2html() because CLI_tutorial.Rmd appears to be an R Markdown v2 document.
Quitting from lines 178-195 (CLI_tutorial.Rmd) 
Error: processing vignette 'CLI_tutorial.Rmd' failed with diagnostics:
there is no package called 'webshot'
Execution halted


checking R code for possible problems ... NOTE
hchart.survfit: no visible binding for global variable ‘group’
Undefined global functions or variables:
  group
```

## PubBias (1.0)
Maintainer: Simon Thornley <sithor@gmail.com>

0 errors | 0 warnings | 3 notes

```
checking DESCRIPTION meta-information ... NOTE
Malformed Title field: should not end in a period.

checking dependencies in R code ... NOTE
Packages in Depends field not imported from:
  ‘R.utils’ ‘rmeta’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.

checking R code for possible problems ... NOTE
ChisqTest_expect: no visible global function definition for ‘pchisq’
plot_chase_observed_expected: no visible global function definition for
  ‘meta.MH’
plot_chase_observed_expected: no visible global function definition for
  ‘txtProgressBar’
plot_chase_observed_expected: no visible global function definition for
  ‘setTxtProgressBar’
test.one.treated: no visible global function definition for
  ‘fisher.test’
test_one: no visible global function definition for ‘rbinom’
test_one: no visible global function definition for ‘fisher.test’
Undefined global functions or variables:
  fisher.test meta.MH pchisq rbinom setTxtProgressBar txtProgressBar
Consider adding
  importFrom("stats", "fisher.test", "pchisq", "rbinom")
  importFrom("utils", "setTxtProgressBar", "txtProgressBar")
to your NAMESPACE file.
```

## QDNAseq (1.10.0)
Maintainer: Daoud Sie <d.sie@vumc.nl>  
Bug reports: https://github.com/ccagc/QDNAseq/issues

0 errors | 0 warnings | 0 notes

## qrcode (0.1.1)
Maintainer: Victor Teh <victorteh@gmail.com>

0 errors | 0 warnings | 0 notes

## R.cache (0.12.0)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/R.cache/issues

0 errors | 0 warnings | 0 notes

## R.devices (2.14.0)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/R.devices/issues

0 errors | 0 warnings | 0 notes

## ReporteRs (0.8.7)
Maintainer: David Gohel <david.gohel@lysis-consultants.fr>  
Bug reports: https://github.com/davidgohel/ReporteRs/issues

0 errors | 0 warnings | 0 notes

## ReportingTools (2.14.0)
Maintainer: Jason A. Hackney <hackney.jason@gene.com>, Gabriel Becker
 <becker.gabe@gene.com>, Jessica L. Larson <larson.jessica@gmail.com>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
.DGEExact.to.data.frame: no visible global function definition for
  'p.adjust'
.DGEExact.to.html: no visible global function definition for 'p.adjust'
.DGELRT.to.data.frame: no visible global function definition for
  'p.adjust'
.DGELRT.to.html: no visible global function definition for 'p.adjust'
.GOhyperG.to.html: no visible global function definition for 'png'
.GOhyperG.to.html: no visible global function definition for 'dev.off'
.GOhyperG.to.html: no visible global function definition for 'pdf'
... 81 lines ...
Undefined global functions or variables:
  HTML cairo_pdf colorRampPalette description dev.off div exprs fData
  featureNames h1 htmlRep mcols p.adjust pdf plot png promptData rect
  runif sessionInfo setNames svg tag tagList tags text write.csv
Consider adding
  importFrom("grDevices", "cairo_pdf", "colorRampPalette", "dev.off",
             "pdf", "png", "svg")
  importFrom("graphics", "plot", "rect", "text")
  importFrom("stats", "p.adjust", "runif", "setNames")
  importFrom("utils", "promptData", "sessionInfo", "write.csv")
to your NAMESPACE file.
```

## R.filesets (2.10.0)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/R.filesets/issues

0 errors | 0 warnings | 0 notes

## R.huge (0.9.0)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/R.huge/issues

0 errors | 0 warnings | 0 notes

## R.matlab (3.6.1)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/R.matlab/issues

0 errors | 0 warnings | 0 notes

## R.rsp (0.30.0)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/R.rsp/issues

0 errors | 0 warnings | 0 notes

## rvg (0.1.1)
Maintainer: David Gohel <david.gohel@ardata.fr>  
Bug reports: https://github.com/davidgohel/rvg/issues

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  5.8Mb
  sub-directories of 1Mb or more:
    libs   5.6Mb
```

## scphaser (1.0.0)
Maintainer: Daniel Edsgärd <edsgard@hotmail.com>  
Bug reports: https://github.com/edsgard/scphaser/issues

0 errors | 0 warnings | 0 notes

## SDMTools (1.1-221)
Maintainer: Jeremy VanDerWal <jjvanderwal@gmail.com>

0 errors | 0 warnings | 2 notes

```
checking dependencies in R code ... NOTE
'library' or 'require' calls in package code:
  ‘raster’ ‘sp’
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.

checking R code for possible problems ... NOTE
COGravity: no visible global function definition for ‘sd’
ClassStat: no visible global function definition for ‘na.omit’
ClassStat: no visible global function definition for ‘sd’
ConnCompLabel: no visible global function definition for ‘setValues’
ImageDiff: no visible global function definition for ‘terrain.colors’
ImageDiff: no visible global function definition for ‘image’
PatchStat: no visible global function definition for ‘na.omit’
Scalebar: no visible global function definition for ‘rect’
Scalebar: no visible global function definition for ‘segments’
... 35 lines ...
Undefined global functions or variables:
  CRS GridTopology SpatialGridDataFrame aggregate contour getValues
  gray heat.colors image na.omit pnorm polygon projection<- quantile
  rainbow raster rect sd segments setValues terrain.colors text
Consider adding
  importFrom("grDevices", "gray", "heat.colors", "rainbow",
             "terrain.colors")
  importFrom("graphics", "contour", "image", "polygon", "rect",
             "segments", "text")
  importFrom("stats", "aggregate", "na.omit", "pnorm", "quantile", "sd")
to your NAMESPACE file.
```

## simPATHy (0.2)
Maintainer: Elisa Salviato <elisa.salviato.88@gmail.com>

0 errors | 0 warnings | 0 notes

## snplist (0.16)
Maintainer: Alexander Sibley <alexander.sibley@dm.duke.edu>

0 errors | 0 warnings | 0 notes

## SpaDES (1.3.1)
Maintainer: Alex M Chubaty <alexander.chubaty@canada.ca>  
Bug reports: https://github.com/PredictiveEcology/SpaDES/issues

0 errors | 2 warnings | 1 note 

```
checking whether package ‘SpaDES’ can be installed ... WARNING
Found the following significant warnings:
  Warning: no DISPLAY variable so Tk is not available
See ‘/home/hb/repositories/R.utils/revdep/checks/SpaDES.Rcheck/00install.out’ for details.

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

    RFoptions

The following object is masked from 'package:raster':

    atan2

... 8 lines ...

Attaching package: 'grid'

The following object is masked from 'package:SpaDES':

    gpar

Quitting from lines 352-356 (ii-modules.Rmd) 
Error: processing vignette 'ii-modules.Rmd' failed with diagnostics:
there is no package called 'webshot'
Execution halted

checking package dependencies ... NOTE
Package suggested but not available for checking: ‘fastshp’
```

## spm12r (2.0)
Maintainer: John Muschelli <muschellij2@gmail.com>

0 errors | 0 warnings | 0 notes

## stplanr (0.1.5)
Maintainer: Robin Lovelace <rob00x@gmail.com>  
Bug reports: https://github.com/ropensci/stplanr/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning: It seems you should call rmarkdown::render() instead of knitr::knit2html() because introducing-stplanr.Rmd appears to be an R Markdown v2 document.
trying URL 'http://www.google.com'
Content type 'text/html; charset=ISO-8859-1' length unknown
.......... 
downloaded 10 KB

Quitting from lines 208-219 (introducing-stplanr.Rmd) 
Error: processing vignette 'introducing-stplanr.Rmd' failed with diagnostics:
there is no package called 'webshot'
Execution halted

```

## TBSSurvival (1.2)
Maintainer: Cassio de Campos <cassiopc@acm.org>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
.dunif.exp: no visible global function definition for ‘dunif’
.dunif.exp: no visible global function definition for ‘dexp’
.logpost: no visible global function definition for ‘dnorm’
.pt2: no visible global function definition for ‘pbeta’
.qt2: no visible global function definition for ‘qbeta’
.rlogis2: no visible global function definition for ‘runif’
.rt2: no visible global function definition for ‘runif’
.tbs.survreg: no visible global function definition for ‘model.frame’
.tbs.survreg: no visible global function definition for ‘model.matrix’
... 61 lines ...
  ppoints qbeta qcauchy qlogis qnorm qqline qqplot quantile rcauchy
  rlogis rnorm runif sd ts
Consider adding
  importFrom("graphics", "hist", "lines", "par", "plot")
  importFrom("stats", "acf", "dcauchy", "dexp", "dlogis", "dnorm",
             "dunif", "model.frame", "model.matrix", "model.response",
             "optim", "pbeta", "pcauchy", "plogis", "pnorm", "ppoints",
             "qbeta", "qcauchy", "qlogis", "qnorm", "qqline", "qqplot",
             "quantile", "rcauchy", "rlogis", "rnorm", "runif", "sd",
             "ts")
to your NAMESPACE file.
```

## TCGAbiolinks (2.2.3)
Maintainer: Antonio Colaprico <antonio.colaprico@ulb.ac.be>,
 Tiago Chedraoui Silva <tiagochst@usp.br>  
Bug reports: https://github.com/BioinformaticsFMRP/TCGAbiolinks/issues

0 errors | 0 warnings | 3 notes

```
checking installed package size ... NOTE
  installed size is 19.6Mb
  sub-directories of 1Mb or more:
    R      1.1Mb
    data   6.2Mb
    doc   12.1Mb

checking R code for possible problems ... NOTE
GDCquery_clinic: no visible binding for global variable ‘portions’
TCGAanalyze_DMR: no visible global function definition for ‘as’
TCGAanalyze_Preprocessing: no visible global function definition for
  ‘assays’
TCGAvisualize_oncoprint: no visible binding for global variable ‘value’
TCGAvisualize_starburst: no visible global function definition for ‘as’
checkTumorInput: no visible binding for global variable ‘project’
get.mutation.matrix: no visible binding for global variable ‘Tumor’
update.clinical.with.last.followup: no visible binding for global
  variable ‘bcr_patient_barcode’
Undefined global functions or variables:
  Tumor as assays bcr_patient_barcode portions project value
Consider adding
  importFrom("methods", "as")
to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
contains 'methods').

checking Rd line widths ... NOTE
Rd file 'TCGAvisualize_starburst.Rd':
  \examples lines wider than 100 characters:
     SummarizedExperiment::rowRanges(met)$diffmean.g2.g1 <- -1*(SummarizedExperiment::rowRanges(met)$diffmean.g1.g2)

These lines will be truncated in the PDF manual.
```

## tmle.npvi (0.10.0)
Maintainer: Pierre Neuvial <pierre.neuvial@genopole.cnrs.fr>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
NPVI: no visible global function definition for ‘quantile’
SL.glm.condExpX2givenW: no visible global function definition for
  ‘as.formula’
SL.glm.condExpX2givenW: no visible global function definition for ‘glm’
SL.glm.condExpXYgivenW: no visible global function definition for
  ‘as.formula’
SL.glm.condExpXYgivenW: no visible global function definition for ‘glm’
SL.glm.g: no visible global function definition for ‘as.formula’
SL.glm.g: no visible global function definition for ‘glm’
... 55 lines ...
Undefined global functions or variables:
  as.formula binomial dnorm gaussian glm head integrate model.matrix
  optimize plogis pnorm qlogis qnorm quantile rnorm runif sd str tail
  update var
Consider adding
  importFrom("stats", "as.formula", "binomial", "dnorm", "gaussian",
             "glm", "integrate", "model.matrix", "optimize", "plogis",
             "pnorm", "qlogis", "qnorm", "quantile", "rnorm", "runif",
             "sd", "update", "var")
  importFrom("utils", "head", "str", "tail")
to your NAMESPACE file.
```

## ToPASeq (1.8.0)
Maintainer: Ivana Ihnatova <ihnatova@iba.muni.cz>

0 errors | 0 warnings | 5 notes

```
checking for hidden files and directories ... NOTE
Found the following hidden files and directories:
  .BBSoptions
These were most likely included in error. See section ‘Package
structure’ in the ‘Writing R Extensions’ manual.

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘Rplots.pdf’

checking dependencies in R code ... NOTE
'library' or 'require' calls in package code:
  ‘DEGraph’ ‘plotrix’
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.

checking R code for possible problems ... NOTE
.hote: no visible global function definition for ‘cov’
.hote: no visible global function definition for ‘pf’
.hotePaired: no visible global function definition for ‘cov’
.runCliqueVarTest: no visible global function definition for ‘pchisq’
.runPathwayVarTest: no visible global function definition for ‘qchisq’
.runPathwayVarTest: no visible global function definition for ‘pchisq’
CLIPPER: no visible global function definition for ‘p.adjust’
CLIPPERfast: no visible global function definition for ‘p.adjust’
CMT: no visible global function definition for ‘pf’
... 103 lines ...
Consider adding
  importFrom("grDevices", "colorRampPalette")
  importFrom("graphics", "abline", "arrows", "layout", "legend", "par",
             "plot.new", "plot.window", "points", "polygon", "strheight",
             "strwidth", "symbols", "text", "title")
  importFrom("stats", "cor", "cov", "density", "median", "model.matrix",
             "na.omit", "p.adjust", "pchisq", "pf", "phyper", "pnorm",
             "qchisq", "qnorm", "rbinom", "rnorm", "sd", "setNames",
             "t.test")
  importFrom("utils", "combn", "installed.packages")
to your NAMESPACE file.

checking Rd line widths ... NOTE
Rd file 'AdjacencyMatrix2Pathway.Rd':
  \usage lines wider than 90 characters:
     AdjacencyMatrix2Pathway(adjmat, name = "pathway", ident = "unknown", database = "unknown", species = "unknown", date = NULL)

Rd file 'Clipper.Rd':
  \usage lines wider than 90 characters:
     clipper(x, group, pathways, type, preparePaths=TRUE, norm.method=NULL, test.method=NULL,  method="mean", testCliques=FALSE, nperm=1000, ... [TRUNCATED]
        both.directions=TRUE, maxNodes=150, minEdges=0, commonTh=2, filterSPIA=FALSE, convertTo="none", convertBy=NULL)

... 45 lines ...
Rd file 'preparePathways.Rd':
  \usage lines wider than 90 characters:
     preparePathways(pathways, method, both.directions, genes, maxNodes = 150, minEdges = 0, commonTh = 2, filterSPIA = FALSE, convertTo = " ... [TRUNCATED]

Rd file 'reduceRraph.Rd':
  \examples lines wider than 100 characters:
     nA<-list(fillcolor=c(NRAS="red", KRAS="red", HRAS="red", SHC1="green", SHC4="green", SHC2="green", SHC3="green"))
     plot(as(pathways,"graphNEL"), nodeAttrs=nA, attrs=list(node=list(fontsize=30, height=40)), main="Before")
      nodeAttrs=list(fillcolor=c(RAS="red", SHC="green")), attrs=list(node=list(fontsize=30, height=40)), main="After")

These lines will be truncated in the PDF manual.
```

## Uniquorn (1.2.0)
Maintainer: 'Raik Otto' <raik.otto@hu-berlin.de>

0 errors | 0 warnings | 2 notes

```
checking DESCRIPTION meta-information ... NOTE
Malformed Description field: should contain one or more complete sentences.

checking top-level files ... NOTE
Non-standard file/directory found at top level:
  ‘unitTests’
```

